---
title: "Longitudinal progression of grey matter atrophy in non-amnestic Alzheimer's disease"
author: "\\*Fulvio Da Re, MD^1,2,3^, \\*Jeffrey S. Phillips, PhD^1,4^, David J. Irwin, MD^1,4^, Corey T. McMillan, PhD^1,4^, Sanjeev N. Vaishnavi, MD, PhD^4^, Sharon X. Xie, PhD^5^, Edward B. Lee, MD, PhD^6^, Philip A. Cook, PhD^7^, James C. Gee, PhD^7^, Leslie M. Shaw, PhD^6^, John Q. Trojanowski, MD, PhD^6^, David A. Wolk, MD^4,8^ and Murray Grossman, MD, EdD^1,4^"
date: '`r format(Sys.Date(), "%B %d, %Y")`'
header-includes:
  \usepackage{graphicx}
  \usepackage{morefloats}
  \usepackage{caption}
  \usepackage{lscape}
  \usepackage{siunitx}
  \usepackage{xcolor, colortbl}
  \newcommand{\blandscape}{\begin{landscape}}
  \newcommand{\elandscape}{\end{landscape}}
output:
  pdf_document:
    fig_caption: yes
    keep_tex: yes
    number_sections: yes
  html_document:
    fig_caption: yes
    force_captions: yes
    highlight: pygments
    number_sections: yes
    theme: cerulean
bibliography: naAD_long.bib
csl: brain.csl
---

\pagenumbering{gobble}

\* These authors contributed equally to this work.

__Short title:__ Longitudinal Atrophy in Non-Amnestic AD

__Affiliations:__

^1^Penn Frontotemporal Degeneration Center, University of Pennsylvania, Philadelphia, PA, USA, ^2^PhD Program in Neuroscience, University of Milano-Bicocca, Milan, Italy, ^3^School of Medicine and Surgery, Milan Center for Neuroscience (NeuroMI), ^4^Department of Neurology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA, University of Milano-Bicocca, Milan, Italy, ^5^Department of Biostatistics, Epidemiology, and Informatics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA, ^6^Center for Neurodegenerative Disease Research, University of Pennsylvania, Philadelphia, PA, USA, ^7^Penn Image Computing and Science Laboratory, Department of Radiology, University of Pennsylvania, Philadelphia, PA, USA, ^8^Penn Memory Center, University of Pennsylvania, Philadelphia, PA, USA

__Corresponding author:__

Jeffrey S. Phillips, PhD  
jefphi@pennmedicine.upenn.edu  
Penn Frontotemporal Degeneration Center  
Department of Neurology  
University of Pennsylvania  
3400 Spruce St., 3 Gibson  
Philadelphia, PA 19104

\newpage

\pagenumbering{arabic}

# Abstract {-}

Non-amnestic Alzheimer's disease is associated with domain-specific cognitive deficits and spared episodic memory. However, longitudinal anatomical change in this condition remains understudied. We investigated whether non-amnestic Alzheimer's disease patients exhibit focal neocortical atrophy that spreads longitudinally depending on patients' phenotype. Additionally, we asked whether atrophy spreads more rapidly throughout neocortical regions and more slowly in the medial temporal lobes in non-amnestic than amnestic Alzheimer's disease. We analyzed longitudinal atrophy in 50 non-amnestic patients---including logopenic-variant primary progressive aphasia (n=25), posterior cortical atrophy (n=15), and frontal-variant Alzheimer's disease (n=10)---and 15 amnestic patients, diagnosed by published clinical research criteria and matched for age, disease severity, and disease duration at initial MRI. Patients were compared to 37 demographically-matched controls. All patients had autopsy (n=8) or cerebrospinal fluid (n=57) evidence of Alzheimer's disease pathology. We compared grey matter volume at initial MRI and longitudinal change in regions associated with each phenotype, adjusting for age and severity of cognitive impairment. We observed unique patterns of longitudinal neocortical disease spread in each non-amnestic phenotype. Non-amnestic patients had more gradual atrophy than amnestic patients in right temporoparietal areas. Additionally, logopenic-variant patients exhibited both greater initial atrophy and more rapid longitudinal change in left lateral temporal areas than other groups. Frontal-variant patients had more rapid frontal and slower temporoparietal atrophy than other non-amnestic groups. At the time of initial scan, logopenic-variant and frontal-variant patients had less atrophy than amnestic patients in the medial temporal lobes, but longitudinal atrophy rates in this area did not differ between non-amnestic and amnestic patients. Older age was associated with medial temporal atrophy independent of group. Volume change in phenotype-specific regions of interest was associated with longitudinal performance on measures of episodic memory, language, visuospatial function, and executive function. The distribution and rate of neocortical atrophy differed by phenotype in non-amnestic Alzheimer's disease; these anatomic patterns differ from amnestic patients and may explain phenotype-specific cognitive change. Medial temporal sparing in non-amnestic Alzheimer's disease may be due to later onset of medial temporal degeneration than in the amnestic syndrome rather than different rates of atrophy over time. Despite this relative early sparing, non-amnestic patients were as susceptible as amnestic patients to normal, age-related atrophy of the medial temporal lobes.

# Keywords {-}
Non-amnestic Alzheimer's disease, logopenic-variant primary progressive aphasia, posterior cortical atrophy, frontal-variant Alzheimer's disease, longitudinal progression

# Abbreviations {-}
A$\beta$ = $\beta$-amyloid; aAD= amnestic Alzheimer's disease; naAD = non-amnestic Alzheimer's disease; CBS = corticobasal syndrome; fvAD = frontal-variant Alzheimer's disease; lvPPA = logopenic-variant primary progressive aphasia; PCA = posterior cortical atrophy; GM = grey matter; MTL = medial temporal lobes; ROI = region of interest; MMSE = Mini Mental Status Exam; LME = linear mixed effects; PVLT = Philadelphia Verbal Learning Test; PBAC = Philadelphia Brief Assessment of Cognition

\newpage

# Introduction {-}
Recent observations suggest that non-amnestic syndromes with underlying Alzheimer's disease pathology are more prevalent than previously thought [@dickerson_approach_2017;@peter_subgroups_2014]. These non-amnestic syndromes are associated with a combination of prominent neocortical disease and relative hippocampal sparing [@galton_atypical_2000;@mesulam_primary_2014;@murray_neuropathologically_2011;@whitwell_neuroimaging_2012]. Moreover, patients with atypical disease distributions may have different rates of clinical progression than typical amnestic Alzheimer's disease (aAD) patients [@byun_heterogeneity_2015;@duara_regional_2013;@poulakis_heterogeneous_2018]. However, relatively little research has examined the longitudinal anatomical spread of disease in non-amnestic Alzheimer's disease (naAD). In a previous MRI analysis [@phillips_neocortical_2018], we inferred patterns of atrophy spread in four naAD phenotypes: logopenic-variant primary progressive aphasia (lvPPA), characterized by primary language deficits [@gorno-tempini_classification_2011]; posterior cortical atrophy (PCA), characterized by visuospatial deficits [@crutch_consensus_2017]; frontal-variant Alzheimer's disease (fvAD), defined by deficits in executive function and/or social behavior [@dubois_advancing_2014]; and corticobasal syndrome (CBS), which can present with a constellation of lateralized motor and cognitive deficits [@medaglia_brain_2017]. These results corroborated the hypothesis that each naAD phenotype has a distinct neocortical origin with relative sparing of the medial temporal lobes (MTL). Moreover, this study suggested that each naAD phenotype has a distinct pattern of disease spread that differs from aAD. However, this study used a cross-sectional design and did not directly compare anatomical disease progression between aAD and naAD phenotypes.

In the present study, we used longitudinal MRI to investigate whether aAD and naAD differ in longitudinal rate and anatomic distribution of grey matter (GM) atrophy over time. We reasoned that differences in the longitudinal rate and distribution of degeneration could explain differences in clinical progression across amnestic and non-amnestic Alzheimer's disease variants. In a hypothesis-driven analysis based on anatomic areas implicated in our prior work [@phillips_neocortical_2018], we investigated differences in GM volume at the time of initial MRI as well as volume change over time in regions-of-interest (ROIs) associated with lvPPA, PCA, fvAD, and aAD. Additionally, we performed an exploratory voxelwise analysis of cortical thickness to identify group differences not captured by these ROIs. We sought to identify group differences in atrophy distribution and progression independent of age, which has been previously reported to differ between typical and atypical forms of Alzheimer's disease [@murray_neuropathologically_2011]. Based on the high neocortical disease burden and domain-specific cognitive deficits that we previously observed in naAD, we predicted that naAD patients would exhibit faster rates of atrophy in phenotype-specific neocortical ROIs relative to aAD. Additionally, we tested the hypothesis that naAD patients would exhibit slower atrophy than aAD patients in the hippocampus and surrounding MTL areas, as a possible explanation for the relative memory sparing associated with these structures in naAD.

# Materials and methods {-}
```{r setup, include=FALSE,echo=FALSE}
knitr::opts_chunk$set(fig.path='fig/', warning=FALSE, message=FALSE)
library(knitcitations); cleanbib()
#cite_options(citation_format = "pandoc", cite.style='numeric', check.entries=FALSE)
cite_options(citation_format = "pandoc", cite.style='numeric',
	super=TRUE, style='text',check.entries=FALSE)
library(ANTsR)
library(nlme)
library(ggplot2)
library(doBy)
library(multcomp)
library(plyr)
library(tableone)
library(xtable)
library(grid)
library(ggsignif)
library(gridExtra)

source('~/R/rmdfunx.R')

# Plot color scheme.
gg_color_hue <- function(n) {
  hues = seq(15, 375, length=n+1)
  hcl(h=hues, l=65, c=100)[1:n]
}

wd<-'/Users/jeffreyphillips/Longitudinal'
if (!dir.exists(wd)) { wd<-'/home/jeff/Longitudinal' }
setwd(wd)

```

```{r data_prep1, eval=TRUE, echo=FALSE, include=FALSE, results='asis',autodep=TRUE}

# Missing ICV values for 121952_20170405, 121952_20180313, 122018_20170406, 
# 122018_20180405, 119334_20150908, 119334_20180215

flist<-Sys.glob('pd2018ctx/*CorticalThickness_LabelStats.csv')
lstat<-data.frame()
for (f in flist) {
	s<-unlist(strsplit(basename(f),'_'))
	tmp<-read.csv(f,stringsAsFactors=F)
	tmp$Subj<-s[1];tmp$MRISession<-s[2];tmp$ID<-paste(s[1:2],collapse='_')
	lstat<-rbind.fill(lstat,tmp)
}
means<-subset(lstat,lstat$Type %in% 'Mean')
vols<-subset(lstat,lstat$Type %in% 'Volume')
icv<-read.csv('pd2018_icv.csv',stringsAsFactors=F)
vols$ICV<-icv$ICV[match(vols$ID,icv$ID)]
vols$NormVol<-vols$Value/vols$ICV
df<-read.csv('longitudinal_20180507_df.csv',stringsAsFactors=F)
df$Group2<-factor(df$Group2,levels=c('Control','aAD','lvPPA','PCA','fvAD'))
df$MRISession[df$Subj==119288 & df$MRISession==20160823]<-20160824
df$ID[df$Subj==119288 & df$MRISession==20160824]<-'119288_20160824'
df<-subset(df,!(duplicated(df$INDDID) & df$time_diff<0.5))
df<-subset(df,df$INDDID %in% df$INDDID[duplicated(df$INDDID)])
apoe<-read.csv('JeffAPOE_14May2018.csv',stringsAsFactors=F)
apoe$APOE[apoe$APOE=='']<-NA
df$APOE<-apoe$APOE[match(df$INDDID,apoe$INDDID)]
df$e4<-df$e3<-df$e2<-NA
for (i in 1:nrow(df)) {
	if (!is.na(df$APOE[i])) {
		s<-unlist(strsplit(df$APOE[i],'/'))
		df$e2[i]<-length(which(s=='E2'))
		df$e3[i]<-length(which(s=='E3'))
		df$e4[i]<-length(which(s=='E4'))
	}
}
df$e4Carrier<-ifelse(df$e4>0,TRUE,FALSE)
#stripchart(APOE~Group3,data=df,vertical=T,method='jitter',pch=2)

convol<-subset(vols,vols$Subj %in% df$INDDID[df$Group %in% 'g0_Control' &
	df$time_diff==0])
consumm<-summaryBy(NormVol~Label.Number,data=convol,FUN=c(mean,sd),na.rm=T)
vols$ZVol<-(vols$NormVol-consumm$NormVol.mean[match(vols$Label.Number,
	consumm$Label.Number)])/consumm$NormVol.sd[match(vols$Label.Number,
	consumm$Label.Number)]

roi<-read.csv(file='nba_roi_phases.csv',stringsAsFactors=F)
roi$Group<-gsub('bvAD','fvAD',roi$Group)
p1<-subset(roi,roi$NPStage==1 & ! roi$Group %in% 'CBS')
cxlabs<-sort(c(unique(p1$LabelID[p1$LabelID>60]),116:117,170:171))
cxlabs<-setdiff(cxlabs,c(195,200))
subcxlabs<-c(31:32,36:37,47:48,55:60)

for (i in c(cxlabs,subcxlabs)) {
	df[,ncol(df)+1]<-NA
	labnum<-paste('lab',sprintf('%03d',i),sep='')
	names(df)[ncol(df)]<-labnum
	subvols<-subset(vols,vols$Label.Number==i)
	df[,labnum]<-subvols$ZVol[match(df$ID,subvols$ID)]
}

cxgm<-subset(lstat,lstat$Label.AbbrevName=='Cortical GM' & lstat$Type==
	'NormalizedVolume')
df$CxVol<-cxgm$Value[match(df$ID,cxgm$ID)]

save(file='long_20180503.RData',list=c('df','lstat','means','vols','labs',
	'cxlabs','subcxlabs','roi','p1'))

```

```{r data_prep2, eval=TRUE, echo=FALSE, include=FALSE, results='asis',autodep=TRUE}

knitr::clean_cache(clean=T)
load('long_20180503.RData')
labs<-sort(c(cxlabs,subcxlabs))
hyplabs<-c(47:48, cxlabs)

# Subject-specific adjustments/corrections:
## Exclude 3 control scans so that no invididual contributes more than 4 scans.
df<-subset(df,!df$ID %in% c('101162_20100129','101162_20110309',
	'108853_20090317','108853_20090429','108853_20110713','117964_20140212'))

# Found some more control scans that are less than 6 months apart.
df<-subset(df,!df$ID %in% c('100190_20080813','100620_20100707','105521_20100317',
	'108853_20100303','112932_20100412','113385_20100413','116543_20141003',
	'118053_20140603','118458_20140929','119108_20150709','119159_20150707'))

## Missing disease duration info.
df$base_disdur[df$INDDID==116810]<-2.22

## Exclude case with recent autopsy evidence of FTLD-TDP.
df<-subset(df,!df$INDDID %in% 116569)

# One fvAD case has both Luminex and Elisa CSF; the results are inconsistent but
# lean toward non-AD.
df<-subset(df,!df$INDDID %in% 107603)

# Subjects 110006 (aAD), 110295 (lvPPA), 111073 (fvAD), and 112013 (fvAD)
# are amyloid-positive but not tau-positive. Mostly they qualify on the ratio
# score. Are there any other markers (PET, etc.) to support their
# diagnosis?
# Note that 111073 has borderline amyloid (206) and elevated p-tau (62).
# Similarly, 112013 has p-tau=41 and amyloid=199.
# To be conservative, I'll exclude them for now.
df<-subset(df,!df$INDDID %in% c(110006,110295,111073,112013,111006))

# Age of onset is not currently used in analysis but may be helpful to have.
df$AgeatOnset<-df$base_ageatMRI-df$base_disdur
# Set colorscheme
colscheme1<-gg_color_hue(length(unique(df$Group)))
colscheme2<-gg_color_hue(4)
quad<-FALSE

# Birth dates are in mixed format.
tmp<-as.Date(df$DOB,format='%Y-%m-%d')
w<-grep('/',df$DOB)
tmp[w]<-as.Date(df$DOB[w],format='%m/%d/%Y')
df$DOB<-tmp

# MRI dates are also in mixed format.
tmp<-as.Date(df$MRIDate,format='%Y-%m-%d')
w<-grep('/',df$MRIDate)
tmp[w]<-as.Date(df$MRIDate[w],format='%m/%d/%Y')
df$MRIDate<-tmp

df<-subset(df,!(df$base_ageatMRI<50))
df<-subset(df,!df$INDDID %in% 118740)

# Stratify aAD group into early- and late-onset.
df$AgeatOnset<-df$base_ageatMRI-df$base_disdur
df$AgeBin<-ifelse(df$AgeatOnset<65,'Young','Old')
df$AgeBin<-factor(df$AgeBin,levels=c('Young','Old'))
df$Group3<-as.character(df$Group)
df$Group3<-gsub('g3_aAD','g3a_EOAD',df$Group3)
df$Group3[df$Group3=='g3a_EOAD' & df$AgeBin=='Old']<-'g3b_LOAD'

```

## Patients {-}
```{r npsych, eval=TRUE, echo=FALSE, include=FALSE, results='asis',autodep=TRUE}

base<-read.csv('INQuery_Closest_Neuropsych_20180521.csv',stringsAsFactors=F)
dd<-names(base)[grep('Date',names(base))]
vv<-setdiff(names(base)[19:ncol(base)],dd)
base$Group<-df$Group[match(base$INDDID,df$INDDID)]
base<-subset(base,!(base$Group %in% 'g0_Control' | duplicated(base$INDDID)))
#base<-subset(base,!duplicated(base$INDDID))
base<-subset(base,base$ID %in% df$ID)
base$Evidence<-NA
base$Evidence[base$ElisaTTauAbetaRatio_Transformed>=0.34]<-'ELISA'
base$Evidence[base$LuminexTTauAbetaRatio>=0.34]<-'Luminex'
base$Evidence[grepl('Alzheimer',base$NPDx1)]<-'Autopsy'

df$Evidence<-base$Evidence[match(df$INDDID,base$INDDID)]

for (d in dd) {
	base[,d]<-as.Date(base[,d],format='%m/%d/%Y')
	base[,gsub('Date','Diff',d)]<-as.numeric(base$MRIDate-base[,d])/365.25
}

for (v in vv) {
	w<-which(base[,v]==996)
	if (length(w)>0 & !grepl('Time',v)) { base[w,v]<-0 }
	base[,v]<-cleaner(base[,v])
}
avail<-data.frame(V=vv,stringsAsFactors=F)
avail$good<-NA
for (i in 1:nrow(avail)) {
	avail$good[i]<-cleaner(base[,avail$V[i]],prop=T)
}

# Correct pre-2010 PBAC scores due to differences in scoring.
# Speech: pre-2010 range is 0--3, current is 0--4.
# Behavior: pre-2010 scale is 4=not observed to 1=severe. Post-2010 scale is
# 3=not observed to 0=severe.
base$PBACDate<-base$MRIDate-365.25*base$PBACDiff
base$pre2010<-base$PBACDate<as.Date('2010-01-01',format='%Y-%m-%d')
w<-which(base$pre2010)
base$BehavioralScale[w]<-base$BehavioralScale[w]-1
base$Speech[w]<-round(1.333*base$Speech[w],1)

# Where PBAC recognition data are missing, use PVLT recognition.
#base$Recall<-base$VerbalMemoryRecall*1.5
#base$Recognition<-base$VerbalMemoryRecognition/3
#base$MemoryDiff<-base$PBACDiff
base$Recall<-base$PHLCorrectTrial9/9
base$Recognition<-base$PHLRecognitionDiscrimin
base$MemoryDiff<-base$PHLDiff
w<-which(abs(base$MemoryDiff)>1)
base$Recall[w]<-NA
base$Recognition[w]<-NA
#w<-which(is.na(base$Recognition) & !is.na(base$PHLRecognitionDiscrimin))
#base$Recognition[w]<-base$PHLRecognitionDiscrimin[w]
#base$MemoryDiff[w]<-base$PHLDiff[w]
w<-which(is.na(base$Recognition) & !is.na(base$VerbalMemoryRecognition))
base$Recognition[w]<-base$VerbalMemoryRecognition[w]/3
base$MemoryDiff[w]<-base$PBACDiff[w]
#w<-which(is.na(base$Recall) & !is.na(base$PHLCorrectTrial9))
#base$Recall[w]<-base$PHLCorrectTrial9[w]
#base$MemoryDiff[w]<-base$PHLDiff[w]
w<-which(is.na(base$Recall) & !is.na(base$VerbalMemoryRecall))
base$Recall[w]<-base$VerbalMemoryRecall[w]/6
base$MemoryDiff[w]<-base$PBACDiff[w]

cogvar<-c('Recognition','Speech','FLetterFluency','ForwardSpan',
	'ReyCopy','JOLO','BehavioralScale','OralTrails',
	'ReverseSpan')

# Race/ethnicity
df$WhiteNonLatino<-NA
df$WhiteNonLatino[df$Race=='White' & df$Ethnicity=='Non-Latino']<-TRUE
df$WhiteNonLatino[!(df$Race=='White' & df$Ethnicity=='Non-Latino')]<-FALSE

# Calculate inter-scan interval.
w<-which(duplicated(df$INDDID))
df$TimeDiff2<-0
df$TimeDiff2[w]<-df$time_diff[w]-df$time_diff[w-1]

tab1Vars<-c('Sex','Education','base_ageatMRI','MMSETotal',
	'TimeDiff2','base_disdur',cogvar)
catVars<-c('Group2','Sex')
qVars<-setdiff(tab1Vars,catVars)
tab1df<-subset(df,!duplicated(df$Subj))

# Align neuropsych data with Table 1 data frame.
# Combined memory scores.
tmp<-subset(base,abs(base$MemoryDiff)<=1)
tab1df<-cbind(tab1df,tmp[match(tab1df$Subj,tmp$INDDID),c('MemoryDiff',
	'Recognition')])

# Digit span.
tmp<-subset(base,abs(base$DigitSpanDiff)<=1)
tab1df<-cbind(tab1df,tmp[match(tab1df$Subj,tmp$INDDID),c('ForwardSpan',
	'ReverseSpan')])

# PBAC scores.
tmp<-subset(base,abs(base$PBACDiff)<=1)
tab1df<-cbind(tab1df,tmp[match(tab1df$Subj,tmp$INDDID),c('PBACDiff',
	'Speech','ReyCopy','JOLO','BehavioralScale',
	'OralTrails','DigitsBackward')])

# F-letter fluency.
tmp<-subset(base,abs(base$FletDiff)<=1)
tab1df<-cbind(tab1df,tmp[match(tab1df$Subj,tmp$INDDID),c('FletDiff',
	'FLetterFluency')])

for (v in cogvar) { tab1df[,v]<-cleaner(tab1df[,v]) }

# Tests of normality.
for (v in setdiff(qVars,'TimeDiff2')) {
	x<-tab1df[,v]
	shapiro.test(x)
#	png(file=paste('fig/hist_',v,'.png',sep=''))
#		xrange<-max(x,na.rm=T)-min(x,na.rm=T)
#		breakseq<-seq(min(x)-0.1*xrange,max(x)+0.1*xrange,by=xrange/10)
#		hist(x,main=v,breaks=breakseq,xlab=v)
#	dev.off()
}

tab1 <- CreateTableOne(vars = tab1Vars, strata = "Group2" , data = tab1df,
	factorVars = catVars, testExact=T)
tab1p<-print(tab1,nonnormal=qVars,contDigits=1)
cogrows<-paste(cogvar,'(median [IQR])')
tab1p[c('base_disdur (median [IQR])',cogrows),'Control']<-'---'
tab1p[2,]<-gsub(')','\\\\%)',tab1p[2,])
tab1p[4,]<-gsub(')','\\\\%)',tab1p[4,])

# Report mean and standard deviation of inter-scan interval.
rind<-which(rownames(tab1p)=='TimeDiff2 (median [IQR])')
for (g in c('Control','aAD','lvPPA','PCA','fvAD')) {
	s<-summary(df$TimeDiff2[duplicated(df$INDDID) & df$Group2 %in% g],na.rm=T)
	tab1p[rind,g]<-paste(sprintf('%.1f',s[3]),' [',sprintf('%.1f',s[2]),', ',
		sprintf('%.1f',s[5]),']',sep='')
#	m<-mean(df$TimeDiff2[duplicated(df$INDDID) & df$Group2 %in% g],na.rm=T)
#	s<-sd(df$TimeDiff2[duplicated(df$INDDID) & df$Group2 %in% g],na.rm=T)
#	tab1p[rind,g]<-paste(round(m,1),' (',round(s,1),')',sep='')
}
k<-kruskal.test(TimeDiff2~Group2,data=df[duplicated(df$INDDID),])
tab1p[rind,'p']<-round(k$p.value,3)

# Find number of observations for each cognitive test.
for (g in c('aAD','lvPPA','PCA','fvAD')) {
	for (i in 1:length(cogrows)) {
		v<-cogvar[i]
		ngood<-length(tab1df[tab1df$Group2 %in% g & !is.na(tab1df[,v]),v])
		tab1p[cogrows[i],g]<-paste(tab1p[cogrows[i],g],' (',ngood,')',sep='')
	}
}

rownames(tab1p)<-c('N','Male','Education',
	'Age at MRI (years)','MMSE (0--30)','Inter-scan interval (years)','Disease duration (years)',
	'Recognition memory (discrimination, 0--1)','Speech (0--4)','Letter fluency (\\# words/60 s)',
	'Forward digit span (length correct)','Rey figure copy (0--12)',
	'Judgment of line orientation (0--6)','Social behavior (0--18)',
	'Oral trail-making test (0--6)','Backward digit span (length correct)')

# Perform post-hoc comparisons
glist<-sort(unique(tab1df$Group))
npsy_base_phoc<-data.frame()

for (v in c('base_ageatMRI','base_MMSETotal')) {
	tasktmp<-data.frame()
	for (i in 1:(length(glist)-1)) {
		g1<-glist[i]
		for (j in (i+1):length(glist)) {
			g2<-glist[j]
			tmpdf<-subset(tab1df,tab1df$Group %in% c(g1,g2))
			m1<-median(tmpdf[tmpdf$Group==g1,v],na.rm=T)
			m2<-median(tmpdf[tmpdf$Group==g2,v],na.rm=T)
			wt<-wilcox.test(as.formula(paste(v,'~Group')),data=tmpdf,conf.int=T)
			tmp<-data.frame(V=v,G1=g1,G2=g2,Median1=m1,Median2=m2,
				U=wt$statistic,LB=wt$conf.int[1],Estimate=wt$estimate,
				UB=wt$conf.int[2],P=wt$p.value,stringsAsFactors=F)
			tasktmp<-rbind(tasktmp,tmp)
		}
	}
	tasktmp$Padj<-p.adjust(tasktmp$P,'fdr')
	npsy_base_phoc<-rbind(npsy_base_phoc,tasktmp)
}

for (v in cogvar) {
	tasktmp<-data.frame()
	for (i in 2:(length(glist)-1)) {
		g1<-glist[i]
		for (j in (i+1):length(glist)) {
			g2<-glist[j]
			tmpdf<-subset(tab1df,tab1df$Group %in% c(g1,g2))
			m1<-median(tmpdf[tmpdf$Group==g1,v],na.rm=T)
			m2<-median(tmpdf[tmpdf$Group==g2,v],na.rm=T)
			wt<-wilcox.test(as.formula(paste(v,'~Group')),data=tmpdf,conf.int=T)
			tmp<-data.frame(V=v,G1=g1,G2=g2,Median1=m1,Median2=m2,
				U=wt$statistic,LB=wt$conf.int[1],Estimate=wt$estimate,
				UB=wt$conf.int[2],P=wt$p.value,stringsAsFactors=F)
			tasktmp<-rbind(tasktmp,tmp)
		}
	}
	tasktmp$Padj<-p.adjust(tasktmp$P,'fdr')
	npsy_base_phoc<-rbind(npsy_base_phoc,tasktmp)
}

```

```{r print_table1, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

xtab1p<-xtable(tab1p[,-ncol(tab1p)],
	caption='Participant characteristics at time of first scan. Median values and interquartile ranges (square brackets) are given for all continuous variables. Education, disease duration, and age at MRI are expressed in years. For each cognitive score, numbers in parentheses indicate the number of observations per group. P-values reflect the results of a chi-squared test for sex and Kruskal-Wallis tests for all other variables.',
	align=c('l',rep('r',ncol(tab1p)-1)),digits=rep(0,ncol(tab1p)))
xtab1p[,1]<-as.character(xtab1p[,1])
names(xtab1p)[ncol(xtab1p)]<-'P'
print(xtab1p,comment=FALSE,caption.placement='top',floating=T,include.rownames=T,
	sanitize.text.function=identity,scalebox=0.67)

```

Participants were recruited through the Penn Frontotemporal Degeneration Center (FTDC) and the Penn Memory Center (PMC) at the University of Pennsylvania, including 37 elderly controls, 15 patients with aAD, and 50 patients with naAD (25 with lvPPA, 15 with PCA, and 10 with fvAD). All procedures were approved by the University of Pennsylvania's Institutional Review Board, and all patients and/or their caregivers gave written informed consent according to the principles established by the Declaration of Helsinki. All patients were clinically diagnosed by experienced neurologists (MG, DJI, DW, and SV), and diagnoses were confirmed by consensus after patients' initial visit by clinicians with expertise in dementia. All patients were at least 50 years of age and had no major cerebrovascular disease, stroke, head trauma, or comorbid psychiatric, neurodegenerative, medical, or developmental disorders. Clinical criteria for each phenotype were as follows: for aAD, primary memory impairment plus deficits in one or more additional cognitive domains [@mckhann_diagnosis_2011]; for lvPPA, primary language impairment including deficits in repetition and/or naming [@giannini_clinical_2017;@gorno-tempini_classification_2011]; for PCA, visuospatial deficits (e.g., in object/spatial perception, neglect, or oculomotor apraxia) [@crutch_consensus_2017]; and for fvAD, deficits in executive function or social behavior. NaAD patients had relatively preserved abilities in all cognitive domains except their domain of primary impairment at initial presentation. All patients had evidence of Alzheimer's disease pathology based on autopsy results (n=9) or a CSF total tau/beta-amyloid 1--42 ratio greater than 0.34 (n=57) according to methods previously described [@shaw_cerebrospinal_2009;@irwin_comparison_2012]. APOE genotyping was performed on 62 of 66 patients. One patient (white male, aAD, age 51 at onset) with an APOE $\epsilon3$/$\epsilon4$ genotype was found to have a mutation in the PSEN1 gene.

Shapiro-Wilks tests indicated non-normal distributions for education and disease duration, age, and MMSE score at initial MRI (all p<0.05). Kruskal-Wallis tests of group differences were non-significant, with the exception of MMSE [$\chi^2$(4)=56.1, p<0.001], reflecting patients' cognitive deficits relative to controls. Mann-Whitney tests confirmed that all patient groups exhibited significantly lower MMSE scores than controls (all U>=4.0, p<0.001); all other pairwise comparisons were non-significant. To corroborate naAD patients' domain-specific cognitive impairment, we analyzed neuropsychological performance on assessments independent of those used in clinical diagnosis, including performance on specific items of the Philadelphia Brief Assessment of Cognition (PBAC) [@libon_philadelphia_2011]. Language was assessed in terms of speech features (with lower scores indicating speech and language impairment), forward digit span as a measure of repetition [@giannini_clinical_2017], and letter fluency, which is sensitive to deficits in executive-mediated lexical retrieval [@ramanan_longitudinal_2017;@rascovsky_disparate_2007]. Visuospatial function was assessed by patients' ability to copy a modified version of the Rey complex figure as well as the judgment of line orientation. Social behavior was assessed on an 18-point scale evaluating social comportment, apathy, disinhibition, agitation, empathy, and ritualistic behaviors. Executive function was evaluated through an oral version of the trail-making test as well as backward digit span. Finally, episodic memory was assessed by recognition on the Philadelphia Verbal Learning Test (PVLT) [@libon_verbal_2011] or the PBAC verbal memory test, as available. All neuropsychological assessments were acquired within 1 year of the initial MRI scan (PVLT and letter fluency: mean=0.18 years, SD=0.25; PBAC: mean=0.27 years, SD=0.29; digit span: mean=0.16 years, SD=0.32). Results were consistent with each phenotype's primary impairment in all domains except for executive function (Table 1). Post-hoc comparisons between all patient groups are reported in Supplementary Table 1.

Participants were required to have longitudinal T1-weighted structural MRI data. We retrospectively selected 163 T1-weighted anatomical MRI scans from patients and 85 scans from controls; all scans were collected on the same 3.0-Tesla Siemens TIM Trio scanner at the Hospital of the University of Pennsylvania. A majority of participants (39/66 patients and 29/37 controls) had only 2 available scans; the remaining participants contributed 3--4 scans each. We included scans acquired with a minimum inter-scan interval of 6 months up to 3.5 years from the initial MRI; beyond this window, there were insufficient observations for a valid analysis. Due to the limited number of observations, only linear associations between atrophy and time were assessed. MRI scans were screened for signs of cerebrovascular disease, hydrocephalus, or white matter lesions; those with a Fazekas scale score>1 were excluded. The current study included 54 patients from our previous, cross-sectional study [@phillips_neocortical_2018] (aAD, n=9; lvPPA, n=24; PCA, n=15; and fvAD, n=6).

## Neuroimaging methods {-}
T1-weighted MR images were acquired axially with 0.98 mm x 0.98 mm x 1 mm voxels, a 256 x 192 matrix, a repetition time of 1620 ms, an inversion time of 950 ms, and a flip angle of 15$^\circ$. Scans were visually inspected for quality by two authors (JP and FDR). Advanced Normalization Tools (ANTs) [@avants_insight_2014;@tustison_large-scale_2014] was used to process each image using a prior-based approach. Images underwent intensity normalization [@tustison_n4itk:_2010] and were spatially normalized to a template based on healthy controls from the Open Access Series of Imaging Studies (OASIS) dataset [@marcus_open_2007] using a symmetric diffeomorphic algorithm [@avants_reproducible_2011;@klein_evaluation_2009]. Images were segmented into 6 tissue classes (cortical grey matter, subcortical grey matter, deep white matter, CSF, brainstem, and cerebellum) using template-based priors; this tissue segmentation was then used to estimate cortical thickness [@tustison_large-scale_2014]. We used a joint label fusion approach [@wang_multi-atlas_2013] to align the Mindboggle-101 labels (based on the Desikan-Killainy-Tourville label scheme) [@klein_101_2012] with each image using pseudo-geodesic registration [@tustison_explicit_2013] and calculated the volume of GM voxels within each label, normalized by intracranial volume and converted to a z-score relative to controls' initial scans. To perform voxelwise group analyses, we warped cortical thickness images to the template using the previously-computed spatial transforms; these images were then spatially smoothed with a 2-sigma Gaussian kernel and downsampled to 2 mm isotropic voxels.

## Statistical analysis {-}
In a hypothesis-driven analysis, we analyzed GM volumes in phenotype-specific ROIs from our previous study of disease progression in naAD [@phillips_neocortical_2018]. This study identified the regions most commonly atrophied in each naAD phenotype, reflecting the likely anatomical origin of disease. These ROIs included left middle and superior temporal gyri in lvPPA; right precuneus, superior parietal lobule, and angular, supramarginal, and middle temporal gyri in PCA; and left anterior insula and middle frontal gyrus as well as right middle temporal gyrus in fvAD (Table 2). Each ROI was expected to exhibit lower volume at the time of participants' initial MRI scan as well as more rapid volume loss over time in its associated patient group(s) relative to other groups. We additionally hypothesized that the aAD group would demonstrate selective atrophy in the MTL, including bilateral hippocampi, parahippocampal gyri, and entorhinal cortex. Atrophy at the time of initial MRI was analyzed using multiple linear regression models with a factor of group and covariates for age and MMSE score at the time of initial MRI as well as the interval between MMSE administration and MRI; controls formed the reference group in these models. Longitudinal atrophy was assessed using linear mixed effects (LME) models with fixed factors of group, time since first scan, and the interaction of group x time. As in the baseline model, covariates included age and MMSE score at initial MRI as well as the interval between MMSE and initial MRI. A subject-specific random intercept was included to account for intra-individual correlations in imaging measures. Post-hoc comparisons were performed for the effect of group at initial MRI as well as the group x time interaction in longitudinal models; values of p<0.05, FDR-corrected, were considered significant.

We used LME models to relate GM volume change to neuropsychological performance within 1 year of each imaging session. The mean interval between test and MRI was 0.19 years (SD=0.24) for recognition memory; 0.22 years (SD=0.24) for letter fluency; 0.18 years (SD=0.23) for digit span; and 0.19 years (SD=0.24) for all other longitudinal neuropsychological measures. Separate LME models were computed for each measure and change in associated ROIs. Thus, recognition performance was related to GM volume in each of the 6 MTL ROIs; language measures were compared to volume change in left middle and superior temporal gyrus; visuospatial measures were related to change in the right superior parietal lobule, precuneus, and angular, supramarginal, and middle temporal gyri; and behavioral and executive measures were related to left anterior insula and middle frontal gyrus as well as right middle temporal gyrus. Neuropsychological performance formed the outcome in each model; predictors treated as fixed effects included regional GM volume at initial MRI and subsequent volume change. Additionally, a subject-specific random intercept was included in the LME model. Due to limited neuropsychological data, controls were omitted from these models, and lvPPA patients formed the reference group. The association with regional volume change in each model was assessed at a significance level of p<0.05, FDR-corrected. To assess the regional specificity of cognitive associations, we also related each neuropsychological measure to longitudinal change in whole cortical volume. 

Additionally, we performed exploratory voxelwise analysis to investigate differences in cortical thickness that were not assessed by a priori ROIs. ROI-based and voxelwise analyses both present distinct advantages and weaknesses. Voxelwise analysis is not constrained by the borders of anatomically-defined ROIs, and it allows more precise anatomical localization of effects. However, voxelwise parametric tests depend on patients’ displaying neurodegeneration at the same precise point within a brain area. Thus, ROI-based volumetric analysis may be more sensitive to atrophy if the precise focus of atrophy within a region differs across individuals. Voxelwise analysis did not include hippocampus, where cortical thickness is not well estimated, but did include entorhinal cortex and parahippocampal gyri. As in ROI-based analysis, we used multiple regression to assess group differences at initial MRI and an LME model to investigate longitudinal atrophy. These voxelwise models used the same regression formulae as ROI-based models, and the LME was implemented in the 3dLME [@chen_linear_2013] function from the Analysis of Functional NeuroImaging (AFNI) software suite. Multiple comparisons correction was performed by first thresholding voxelwise results at p<0.001 (uncorrected), then applying a cluster extent threshold corresponding to a cluster-wise alpha value of 0.05. To calculate cluster extent thresholds, we first estimated spatial auto-correlation from the model residuals using AFNI's 3dFWHMx. We then used the 3dClustSim function, which is based on a Monte Carlo approach [@cox_fmri_2017;@forman_improved_1995], to determine the cluster size corresponding to a false-positive rate of 0.05 at p<0.001 (uncorrected). These simulations indicated a cluster threshold of 71 voxels (i.e., \SI{568}{\micro\litre}) for the baseline MRI model and a threshold of 70 voxels (\SI{560}{\micro\litre}) for the longitudinal LME model. For both the baseline effect of group and the group x time interaction, we performed post-hoc contrasts between all groups, which were corrected to cluster-wise p<0.05 using the same method. In the Supplementary Material, we additionally display voxelwise contrasts vs. controls at a lenient threshold of p<0.01, uncorrected for multiple comparisons.

## Data and availability {-}
Computer code for the current manuscript (including all text, analysis, and visualization of results) is available in the form of an Rmarkdown script in a public GitHub repository (https://github.com/jeffrey-phillips/naAD-longitudinal.git). This code requires [R](https://cran.r-project.org/) version 3.4.4 or higher. Investigators who wish to access imaging and clinical data may submit a direct request to the corresponding author.

# Results {-}

```{r roi_lme, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

# Prepare contrast matrix for additional linear contrasts.
i<-47
fm<-as.formula(paste('lab',sprintf('%03d',i),
	'~Group*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI',sep=''))
m<-lme(fixed=fm, random=~1|Subj,data=df,na.action=na.exclude)
nc<-length(fixef(m))
gnames<-sort(unique(df$Group))
kmat<-matrix(0,nrow=length(gnames)*(length(gnames)-1),ncol=nc)
kmat[5:7,2]<-kmat[8:9,3]<-kmat[10,4]<-kmat[15:17,10]<-kmat[18:19,11]<-kmat[20,12]<-(-1)
kmat[1,2]<-kmat[2,3]<-kmat[3,4]<-kmat[4,5]<-kmat[5,3]<-kmat[6,4]<-kmat[7,5]<-
	kmat[8,4]<-kmat[9:10,5]<-kmat[11,10]<-kmat[12,11]<-kmat[13,12]<-kmat[14,13]<-
	kmat[15,11]<-kmat[16,12]<-kmat[17,13]<-kmat[18,12]<-kmat[19:20,13]<-1
kmat<-data.matrix(kmat)
rownames(kmat)<-paste(rep(c('mean','time'),each=10),'_',rep(gnames[c(2:5,3:5,4:5,5)],2),
	'-',rep(gnames[c(rep(1,4),rep(2,3),rep(3,2),4)],2),sep='')

# Also contrast aAD vs. all naAD.
df$Group4<-as.character(df$Group)
df$Group4<-gsub('g3_aAD','g2_aAD',df$Group4)
df$Group4[df$Group4 %in% c('g1_lvPPA','g2_PCA','g4_fvAD')]<-'g3_naAD'

kmat4<-kmat
kmat4<-kmat4[c(1:2,5,11:12,15),c(1:3,6:11)]
rownames(kmat4)<-gsub('g1_lvPPA','g1_aAD',rownames(kmat4))
rownames(kmat4)<-gsub('g2_PCA','g2_naAD',rownames(kmat4))

# Prepare matrix of p-values for later multiple comparisons adjustment.
pmat1<-matrix(NA,ncol=length(hyplabs),nrow=7)
anovatab<-data.frame()
ttab<-data.frame()
ttab4<-data.frame()
gmeans<-data.frame()
gtimes<-data.frame()
posthoc<-data.frame()
phoc4<-data.frame()

# Run linear mixed effects models and contrasts on GMP or volume data.
for (ind in 1:length(hyplabs)) {
	i<-hyplabs[ind]
	fm<-as.formula(paste('lab',sprintf('%03d',i),
		'~Group*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI',sep=''))
	fm4<-as.formula(paste('lab',sprintf('%03d',i),
		'~Group4*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI',sep=''))
	m<-lme(fixed=fm, random=~1|Subj,data=df,na.action=na.exclude)
	m4<-lme(fixed=fm4, random=~1|Subj,data=df,na.action=na.exclude)
	am<-anova(m)
	pmat1[,ind]<-anova(m)$"p-value"
	labName<-roi$Label.Name[match(i,roi$LabelID)]
	comp <- glht(m, linfct=kmat)
	scomp<-summary(comp,test=adjusted('none'))
	#scomp$test$pvalues[1:10]<-p.adjust(scomp$test$pvalues[1:10],'fdr')
	#scomp$test$pvalues[11:20]<-p.adjust(scomp$test$pvalues[11:20],'fdr')
	tmp<-data.frame(LabelID=i,Comp=names(scomp$test$pvalues),Z=scomp$test$tstat,
		P=scomp$test$pvalues)
	posthoc<-rbind(posthoc,tmp)
	comp4 <- glht(m4, linfct=kmat4)
	scomp4<-summary(comp4,test=adjusted('none'))
	tmp<-data.frame(LabelID=i,Comp=names(scomp4$test$pvalues),
		Estimate=scomp4$test$coefficients,Z=scomp4$test$tstat,
		P=scomp4$test$pvalues)
	phoc4<-rbind(phoc4,tmp)
	
	fg<-round(am[2,3],1)
	ft<-round(am[3,3],1)
	fgt<-round(am[7,3],1)
	tmp<-data.frame(LabelID=i,Region=labName,Group=fg,Time=ft,GroupTime=fgt,
		stringsAsFactors=F)
	anovatab<-rbind(anovatab,tmp)
	tt<-summary(m)$tTable
	tmp2<-as.data.frame(tt)
	tmp2$Effect<-rownames(tmp2)
	tmp2$LabelID=i; tmp2$Label.Name=labName
	ttab<-rbind(ttab,tmp2)
	tmp3<-data.frame(LabelID=i,Group=c('lvPPA','PCA','aAD','fvAD'),
		Beta=tt[2:5,1],SE=tt[2:5,2])
	gmeans<-rbind(gmeans,tmp3)
	tmp4<-data.frame(LabelID=i,Group=c('lvPPA','PCA','aAD','fvAD'),
		Beta=tt[10:13,1],SE=tt[10:13,2])
	gtimes<-rbind(gtimes,tmp4)
}

gtdf1<-am[7,1]
gtdf2<-am[7,2]

pmat1<-t(pmat1);
padj1<-apply(pmat1,2,function(x) { p.adjust(x,method='fdr') })
posthoc$Comp<-as.character(posthoc$Comp)
w<-grep('mean',posthoc$Comp)
posthoc$P[w]<-p.adjust(posthoc$P[w],'fdr')
w<-grep('time',posthoc$Comp)
posthoc$P[w]<-p.adjust(posthoc$P[w],'fdr')
posthoc$G1<-NA;posthoc$G2<-NA
for (i in 1:nrow(posthoc)) {
	s<-unlist(strsplit(posthoc$Comp[i],'-'))
	posthoc$G1[i]<-unlist(strsplit(s[1],'_'))[3]
	posthoc$G2[i]<-unlist(strsplit(s[2],'_'))[2]
}
posthoc$Label.Name<-roi$Label.Name[match(posthoc$LabelID,roi$LabelID)]
rownames(posthoc)<-1:nrow(posthoc)

gmeans$Group<-factor(gmeans$Group,levels=c('aAD','lvPPA','PCA','fvAD'))
gmeans$Label.Name<-roi$Label.Name[match(gmeans$LabelID,roi$LabelID)]
gmeans$Association<-NA
for (i in 1:nrow(gmeans)) {
	gmeans$Association[i]<-paste(unique(setdiff(p1$Group[p1$Label.Name==gmeans$Label.Name[i]],
		'aAD')),collapse=', ')
}
gmeans$Association[grepl('entorhinal',gmeans$Label.Name) | grepl('hippocamp',
	gmeans$Label.Name,ignore.case=TRUE)]<-'aAD'
multigroup<-setdiff(sort(gmeans$Association),as.character(levels(df$Group2)))
gmeans$Association<-factor(gmeans$Association,levels=c(levels(df$Group2),
	multigroup))
gmeans$ObsLab<-gmeans$Label.Name
gmeans$ObsLab<-tolower(gmeans$ObsLab)
gmeans$ObsLab<-gsub('left','L',gmeans$ObsLab)
gmeans$ObsLab<-gsub('right','R',gmeans$ObsLab)
gmeans$ObsLab<-gsub(' gyrus','',gmeans$ObsLab)
gmeans$ObsLab<-gsub(' area','',gmeans$ObsLab)
for (i in 1:nrow(gmeans)) {
	s<-unlist(strsplit(gmeans$ObsLab[i],'\\s+'))
	if (length(s)>2) {
		gmeans$ObsLab[i]<-paste(s[-2],collapse=' ')
	}
}
gmeans$BaseObsLab<-gsub('L ','',gsub('R ','',gmeans$ObsLab))
gmeans<-gmeans[order(gmeans$Association,gmeans$BaseObsLab,gmeans$ObsLab,
	gmeans$Group),]
gmeans$ObsLab<-factor(gmeans$ObsLab,levels=rev(unique(gmeans$ObsLab)))
#gmeans<-gmeans[order(gmeans$Association,ordervec),]

gtimes$Group<-factor(gtimes$Group,levels=c('aAD','lvPPA','PCA','fvAD'))
gtimes$Label.Name<-roi$Label.Name[match(gtimes$LabelID,roi$LabelID)]
gtimes$Association<-NA
for (i in 1:nrow(gtimes)) {
	gtimes$Association[i]<-paste(unique(setdiff(p1$Group[p1$Label.Name==gtimes$Label.Name[i]],
		'aAD')),collapse=', ')
}
gtimes$Association[grepl('entorhinal',gtimes$Label.Name) | grepl('hippocamp',
	gtimes$Label.Name,ignore.case=TRUE)]<-'aAD'
multigroup<-setdiff(sort(gtimes$Association),as.character(levels(df$Group2)))
gtimes$Association<-factor(gtimes$Association,levels=c(levels(df$Group2),
	multigroup))
gtimes$ObsLab<-gtimes$Label.Name
gtimes$ObsLab<-tolower(gtimes$ObsLab)
gtimes$ObsLab<-gsub('left','L',gtimes$ObsLab)
gtimes$ObsLab<-gsub('right','R',gtimes$ObsLab)
gtimes$ObsLab<-gsub(' gyrus','',gtimes$ObsLab)
gtimes$ObsLab<-gsub(' area','',gtimes$ObsLab)
for (i in 1:nrow(gtimes)) {
	s<-unlist(strsplit(gtimes$ObsLab[i],'\\s+'))
	if (length(s)>2) {
		gtimes$ObsLab[i]<-paste(s[-2],collapse=' ')
	}
}
gtimes$BaseObsLab<-gsub('L ','',gsub('R ','',gtimes$ObsLab))
gtimes<-gtimes[order(gtimes$Association,gtimes$BaseObsLab, gtimes$ObsLab,
	gtimes$Group),]
gtimes$ObsLab<-factor(gtimes$ObsLab,levels=rev(unique(gtimes$ObsLab)))
#gtimes<-gtimes[order(gtimes$Association,ordervec),]
```

```{r base_roi, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

kbase<-kmat[1:10,1:8]
kbase[,6:8]<-0
rownames(kbase)<-gsub('mean','base',rownames(kbase))

# labnum<-paste('lab',sprintf('%03d',hyplabs),sep='')
baseroi<-subset(df,!duplicated(df$INDDID))

# Prepare matrix of p-values for later multiple comparisons adjustment.
pmatbase<-matrix(NA,ncol=length(hyplabs),nrow=5)
baseanova<-data.frame()
basettab<-data.frame()
basemeans<-data.frame()
phocbase<-data.frame()

# Run linear regression models and contrasts on ROI volume data.
for (ind in 1:length(hyplabs)) {
	i<-hyplabs[ind]
	lab<-paste('lab',sprintf('%03d',i),sep='')
	fm<-as.formula(paste(lab,'~Group+base_MMSETotal+base_MMSETimeDiff+
		base_ageatMRI',sep=''))
	m<-lm(fm,data=baseroi)
	comp <- glht(m, linfct=kbase)
	scomp<-summary(comp,test=adjusted('none'))

	# Store anova results.
	labName<-roi$Label.Name[match(i,roi$LabelID)]
	am<-anova(m)
	pmatbase[,ind]<-anova(m)$"Pr(>F)"
	fg<-round(am[1,4],1)
	tmp<-data.frame(LabelID=i,Region=labName,Group=fg,stringsAsFactors=F)
	baseanova<-rbind(baseanova,tmp)

	tmp<-data.frame(LabelID=i,Comp=names(scomp$test$pvalues),Z=scomp$test$tstat,
		P=scomp$test$pvalues)
	phocbase<-rbind(phocbase,tmp)
	tt<-summary(m)$coefficients
	tmp2<-as.data.frame(tt)
	tmp2$Effect<-rownames(tmp2)
	tmp2$LabelID=i; tmp2$Label.Name=labName
	basettab<-rbind(basettab,tmp2)
	tmp3<-data.frame(LabelID=i,Group=c('lvPPA','PCA','aAD','fvAD'),
		Beta=tt[2:5,1],SE=tt[2:5,2])
	basemeans<-rbind(basemeans,tmp3)
}

basedf1<-am[1,1]
basedf2<-am[5,1]

pmatbase<-t(pmatbase);
padjbase<-apply(pmatbase,2,function(x) { p.adjust(x,method='fdr') })
phocbase$P<-p.adjust(phocbase$P,'fdr')
phocbase$Comp<-as.character(phocbase$Comp)
phocbase$G1<-NA;phocbase$G2<-NA
for (i in 1:nrow(phocbase)) {
	s<-unlist(strsplit(phocbase$Comp[i],'-'))
	phocbase$G1[i]<-unlist(strsplit(s[1],'_'))[3]
	phocbase$G2[i]<-unlist(strsplit(s[2],'_'))[2]
}
phocbase$Label.Name<-roi$Label.Name[match(phocbase$LabelID,roi$LabelID)]
rownames(phocbase)<-1:nrow(phocbase)

basemeans$Group<-factor(basemeans$Group,levels=c('aAD','lvPPA','PCA','fvAD'))
basemeans$Label.Name<-roi$Label.Name[match(basemeans$LabelID,roi$LabelID)]
basemeans$Association<-NA
for (i in 1:nrow(basemeans)) {
	basemeans$Association[i]<-paste(unique(setdiff(p1$Group[p1$Label.Name==basemeans$Label.Name[i]],
		'aAD')),collapse=', ')
}
basemeans$Association[grepl('entorhinal',basemeans$Label.Name) | grepl('hippocamp',
	basemeans$Label.Name,ignore.case=TRUE)]<-'aAD'
multigroup<-setdiff(sort(basemeans$Association),as.character(levels(df$Group2)))
basemeans$Association<-factor(basemeans$Association,levels=c(levels(df$Group2),
	multigroup))
basemeans$ObsLab<-basemeans$Label.Name
basemeans$ObsLab<-tolower(basemeans$ObsLab)
basemeans$ObsLab<-gsub('left','L',basemeans$ObsLab)
basemeans$ObsLab<-gsub('right','R',basemeans$ObsLab)
basemeans$ObsLab<-gsub(' gyrus','',basemeans$ObsLab)
basemeans$ObsLab<-gsub(' area','',basemeans$ObsLab)
for (i in 1:nrow(basemeans)) {
	s<-unlist(strsplit(basemeans$ObsLab[i],'\\s+'))
	if (length(s)>2) {
		basemeans$ObsLab[i]<-paste(s[-2],collapse=' ')
	}
}
basemeans$BaseObsLab<-gsub('L ','',gsub('R ','',basemeans$ObsLab))
basemeans<-basemeans[order(basemeans$Association,basemeans$BaseObsLab,basemeans$ObsLab,
	basemeans$Group),]
basemeans$ObsLab<-factor(basemeans$ObsLab,levels=rev(unique(basemeans$ObsLab)))

```

```{r table2,eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

posthoc$I1<-toupper(substr(posthoc$G1,1,1))
posthoc$I1<-factor(posthoc$I1,levels=c('A','L','P','F'))
posthoc$I2<-toupper(substr(posthoc$G2,1,1))
posthoc$I2<-factor(posthoc$I2,levels=c('C','A','L','P'))
posthoc$Code<-ifelse(posthoc$Z>0, paste(posthoc$I1,posthoc$I2,sep='>'),
	paste(posthoc$I1,posthoc$I2,sep='<'))
posthoc<-posthoc[order(posthoc$LabelID,posthoc$I1,posthoc$I2),]

phocbase$I1<-toupper(substr(phocbase$G1,1,1))
phocbase$I1<-factor(phocbase$I1,levels=c('A','L','P','F'))
phocbase$I2<-toupper(substr(phocbase$G2,1,1))
phocbase$I2<-factor(phocbase$I2,levels=c('C','A','L','P'))
phocbase$Code<-ifelse(phocbase$Z>0, paste(phocbase$I1,phocbase$I2,sep='>'),
	paste(phocbase$I1,phocbase$I2,sep='<'))
phocbase<-phocbase[order(phocbase$LabelID,phocbase$I1,phocbase$I2),]

basetmp<-subset(phocbase, phocbase$P<0.05)
meantmp<-subset(posthoc,grepl('mean',posthoc$Comp) & posthoc$P<0.05)
timetmp<-subset(posthoc,grepl('time',posthoc$Comp) & posthoc$P<0.05)
#anovatab$MeanPhoc<-'---'
anovatab$BasePhoc<-'---'
anovatab$TimePhoc<-'---'
glist2<-as.character(levels(df$Group2)[-1])

colorcell<-function(x,col,textcol) { paste("\\cellcolor{",col,"}\\textcolor{",
	textcol,"}{",x,"}",sep='') }

for (g in glist2) {
	basetmp<-subset(phocbase,phocbase$G1 %in% g & phocbase$G2 %in% 'Control')
	basetmp<-basetmp[match(anovatab$LabelID,basetmp$LabelID),]
	basecname<-paste('base',g,sep='_')
	anovatab[,basecname]<-sprintf('%.1f',basetmp$Z)

	longtmp<-subset(posthoc,posthoc$G1 %in% g & posthoc$G2 %in% 'Control' &
		grepl('time_',posthoc$Comp))
	longtmp<-longtmp[match(anovatab$LabelID,longtmp$LabelID),]
	longcname<-paste('time',g,sep='_')
	anovatab[,longcname]<-sprintf('%.1f',longtmp$Z)

	w<-which(basetmp$P<=0.05 & longtmp$P>0.05)
	anovatab[w,basecname]<-colorcell(anovatab[w,basecname],'blue','white')
		
	w<-which(longtmp$P<=0.05 & basetmp$P>0.05)
	anovatab[w,longcname]<-colorcell(anovatab[w,longcname],'red','white')

	w<-which(longtmp$P<=0.05 & basetmp$P<=0.05)
	anovatab[w,basecname]<-colorcell(anovatab[w,basecname],'green','black')
	anovatab[w,longcname]<-colorcell(anovatab[w,longcname],'green','black')
}
	
for (i in 1:nrow(anovatab)) {
	x<-basetmp$Code[basetmp$LabelID==anovatab$LabelID[i]]
	if (length(x)>0) {
		anovatab$BasePhoc[i]<-paste(x,collapse=', ')
	}
	x<-timetmp$Code[timetmp$LabelID==anovatab$LabelID[i]]
	if (length(x)>0) {
		anovatab$TimePhoc[i]<-paste(x,collapse=', ')
	}
}
anovatab2<-anovatab
anovatab2$Region<-tolower(anovatab2$Region)
anovatab2$Region<-gsub('left','L',anovatab2$Region)
anovatab2$Region<-gsub('right','R',anovatab2$Region)
anovatab2$Region<-gsub(' gyrus','',anovatab2$Region)
anovatab2$Region<-gsub(' area','',anovatab2$Region)
for (i in 1:nrow(anovatab2)) {
	s<-unlist(strsplit(anovatab2$Region[i],'\\s+'))
	if (length(s)>2) {
		anovatab2$Region[i]<-paste(s[-2],collapse=' ')
	}
}
anovatab<-anovatab[,c('Region','Group','base_aAD','base_lvPPA','base_PCA',
	'base_fvAD','GroupTime','time_aAD','time_lvPPA','time_PCA','time_fvAD')]
#anovatab$Group<-sprintf('%.1f',anovatab$Group)
anovatab$Group<-sprintf('%.1f',baseanova$Group)
anovatab$GroupTime<-sprintf('%.1f',anovatab$GroupTime)
w05<-which(padj1[,2]<0.05)
anovatab$Group[w05]<-paste(anovatab$Group[w05],'*',sep='')
w01<-which(padj1[,2]<0.01)
anovatab$Group[w01]<-paste(anovatab$Group[w01],'*',sep='')
w001<-which(padj1[,2]<0.001)
anovatab$Group[w001]<-paste(anovatab$Group[w001],'*',sep='')
w05<-which(padj1[,7]<0.05)
anovatab$GroupTime[w05]<-paste(anovatab$GroupTime[w05],'*',sep='')
w01<-which(padj1[,7]<0.01)
anovatab$GroupTime[w01]<-paste(anovatab$GroupTime[w01],'*',sep='')
w001<-which(padj1[,7]<0.001)
anovatab$GroupTime[w001]<-paste(anovatab$GroupTime[w001],'*',sep='')

wsig<-which(padjbase[,2]<0.05 | padj1[,7]<0.05)
gefftab<-anovatab[wsig,]
gefftab<-gefftab[order(gefftab$Region),]
gefftab$Association<-NA
for (i in 1:nrow(gefftab)) {
	gefftab$Association[i]<-paste(unique(setdiff(p1$Group[p1$Label.Name==gefftab$Region[i]],'aAD')),collapse=', ')
}
gefftab$Association[grepl('entorhinal',gefftab$Region) | grepl('hippocamp',
	gefftab$Region,ignore.case=TRUE)]<-'aAD'
gefftab<-gefftab[,c(ncol(gefftab),1:(ncol(gefftab)-1))]
gefftab$Region<-tolower(gefftab$Region)
gefftab$Region<-gsub('left','L',gefftab$Region)
gefftab$Region<-gsub('right','R',gefftab$Region)
gefftab$Region<-gsub(' gyrus','',gefftab$Region)
gefftab$Region<-gsub(' area','',gefftab$Region)
for (i in 1:nrow(gefftab)) {
	s<-unlist(strsplit(gefftab$Region[i],'\\s+'))
	if (length(s)>2) {
		gefftab$Region[i]<-paste(s[-2],collapse=' ')
	}
}
multigroup<-setdiff(sort(gefftab$Association),as.character(levels(df$Group2)))
gefftab$Association<-factor(gefftab$Association,levels=c(levels(df$Group2),
	multigroup))
reg2<-gsub('L ','',gsub('R ','',gefftab$Region))
gefftab<-gefftab[order(gefftab$Association,reg2),]
gefftab$Association<-as.character(gefftab$Association)
gefftab$Association[duplicated(gefftab$Association)]<-''

xgefftab<-xtable(gefftab,
	caption="Differences in grey matter volume at initial MRI and longitudinal atrophy in hypothesis-driven analysis of regional brain volumes, relative to matched controls. Hypotheses included selective atrophy of neocortical areas associated with early disease in naAD\\textsuperscript{10} and of the MTL (hippocampus, entorhinal cortex, and parahippocampal gyrus) in aAD patients. F-statistics indicate the main effect of group at initial MRI scan and the group x time interaction across all scans. Additional columns report z-statistics for pairwise contrasts of each patient group vs. controls. Blue cells indicate significant differences in volume only at initial MRI; red cells indicate significant differences in longitudinal atrophy rates; and green cells indicate differences in both initial volume and longitudinal atrophy, based on a threshold of p<0.05, FDR-corrected. *p<0.05; **p<0.01; ***p<0.001.",
	align='rrlrrrrrrrrrr',digits=rep(0,13))
names(xgefftab)<-c('A priori association','Region',
	paste('$F_{First~MRI}$(',basedf1,',',basedf2,')',sep=''),
	'aAD','lvPPA','PCA','fvAD',
	paste('$F_{Group~x~Time}$(',gtdf1,',',gtdf2,')',sep=''),
	'aAD','lvPPA','PCA','fvAD')
	
print(xgefftab,comment=FALSE,caption.placement='top',floating=T,
	include.rownames=F,sanitize.text.function=identity,
	sanitize.colnames.function=identity,scalebox=0.67)

```

```{r base_diff_fig, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE,fig.align='center',fig.width=7.5,fig.height=9,fig.path='./fig/',fig.cap='Patient group differences at time of initial MRI in normalized volumes for a priori regions of interest. Plotted values are the regression coefficients representing the difference in regional grey matter volume between each patient group and the healthy control reference group, expressed in z-score units. Error bars represent the standard error of each regression coefficient. More negative values on the x-axis indicate greater atrophy. Black brackets indicate significant pairwise group differences (p<0.05 after false discovery rate correction).'}

dodge<-position_dodge(width=0.64)
limits<-aes(ymax=basemeans$Beta+basemeans$SE,ymin=basemeans$Beta-basemeans$SE)
pbase<-ggplot(data=basemeans,aes(x=ObsLab,y=Beta,fill=Group))+
	geom_bar(stat='identity',position=dodge)+geom_errorbar(limits,width=0.3,
		position=dodge,colour='grey50')+coord_flip()+
	theme(axis.text.y=element_text(size=10),axis.text.x=element_text(size=10,
		face='bold'),axis.title.y=element_blank(),axis.title.x=
		element_text(size=10,face='bold'),axis.ticks.y=element_blank(),
		axis.ticks.x=element_blank(),legend.position='top',
		legend.title=element_blank(),plot.title=element_text(size=10,
		face='bold'))+
	labs(y='Normalized volume at initial MRI')
gbbase<-ggplot_build(pbase)
gbbase$layout$coord$clip <- "off"
#gbbase$layout$clip[gbbase$layout$name == "panel"] <- "off"
basetix<-gbbase$layout$panel_params[[1]]$y.major_source
#basetix<-gbbase$layout$panel_ranges[[1]]$y.major_source
basetix<-data.frame(Mid=rep(basetix,each=6),Inc1=rep(c(0.08,-0.08,-0.08,-0.24,-0.24,-0.24),
	length(basetix)),Inc2=rep(c(0.24,0.08,0.24,-0.08,0.08,0.24),length(basetix)))
basetix$X=basetix$Mid+basetix$Inc1
basetix$XEnd<-basetix$Mid+basetix$Inc2
basetix$tip_length<-0.01
basetix$LabelID<-unique(basemeans$LabelID)[rev(basetix$Mid)]
basetix$G1<-rep(c('fvAD','PCA','fvAD','aAD','aAD','fvAD'),length.out=nrow(basetix))
basetix$G2<-rep(c('PCA','lvPPA','lvPPA','lvPPA','PCA','aAD'),length.out=nrow(basetix))
tmp<-subset(phocbase,grepl('base',phocbase$Comp) & !grepl('Control',phocbase$Comp))
basetix$P<-tmp$P[match(paste(basetix$G1,basetix$G2,basetix$LabelID,sep='_'),paste(tmp$G1,
	tmp$G2,tmp$LabelID,sep='_'))]
basetix<-subset(basetix,basetix$P<0.05)
basetix$YEnd<-basetix$Y<-0
for (i in unique(basetix$LabelID)) {
	w<-which(basetix$LabelID==i)
	basetix$YEnd[w]<-basetix$Y[w]<-seq(0,length(w)*0.04,length.out=length(w))+0.04
}
basetix$Annotation<-''
basetix$Association<-basemeans$Association[match(basetix$LabelID,basemeans$LabelID)]
pbase+geom_signif(data=basetix,mapping=aes(xmin=basetix$X,xmax=basetix$XEnd,annotations=basetix$Annotation,
	y_position=basetix$Y,tip_length=basetix$tip_length),manual=T,inherit.aes=F)

```

```{r roi_mean_diffs, eval=FALSE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE,fig.align='center',fig.width=7.5,fig.height=9,fig.path='./fig/',fig.cap='Patient group differences in normalized volumes for a priori regions of interest. Plotted values are the regression coefficients representing the difference in regional grey matter volume between each patient group and the healthy control reference group, expressed in z-score units. Error bars represent the standard error of each regression coefficient. More negative values on the x-axis indicate greater atrophy. Black brackets indicate significant pairwise group differences (p<0.05 after false discovery rate correction).'}

dodge<-position_dodge(width=0.64)
limits<-aes(ymax=gmeans$Beta+gmeans$SE,ymin=gmeans$Beta-gmeans$SE)
p<-ggplot(data=gmeans,aes(x=ObsLab,y=Beta,fill=Group))+
	geom_bar(stat='identity',position=dodge)+geom_errorbar(limits,width=0.3,
		position=dodge,colour='grey50')+coord_flip()+
	theme(axis.text.y=element_text(size=10),axis.text.x=element_text(size=10,
		face='bold'),axis.title.y=element_blank(),axis.title.x=
		element_text(size=10,face='bold'),axis.ticks.y=element_blank(),
		axis.ticks.x=element_blank(),legend.position='top',
		legend.title=element_blank(),plot.title=element_text(size=10,
		face='bold'))+
	labs(y='Normalized volume')
gb<-ggplot_build(p)
gb$layout$coord$clip <- "off"
#gb$layout$clip[gb$layout$name == "panel"] <- "off"
tix<-gb$layout$panel_params[[1]]$y.major_source
tix<-data.frame(Mid=rep(tix,each=6),Inc1=rep(c(0.08,-0.08,-0.08,-0.24,-0.24,-0.24),
	length(tix)),Inc2=rep(c(0.24,0.08,0.24,-0.08,0.08,0.24),length(tix)))
tix$X=tix$Mid+tix$Inc1
tix$XEnd<-tix$Mid+tix$Inc2
tix$tip_length<-0.01
tix$LabelID<-unique(gmeans$LabelID)[rev(tix$Mid)]
tix$G1<-rep(c('fvAD','PCA','fvAD','aAD','aAD','fvAD'),length.out=nrow(tix))
tix$G2<-rep(c('PCA','lvPPA','lvPPA','lvPPA','PCA','aAD'),length.out=nrow(tix))
tmp<-subset(posthoc,grepl('mean',posthoc$Comp) & !grepl('Control',posthoc$Comp))
tix$P<-tmp$P[match(paste(tix$G1,tix$G2,tix$LabelID,sep='_'),paste(tmp$G1,
	tmp$G2,tmp$LabelID,sep='_'))]
tix<-subset(tix,tix$P<0.05)
tix$YEnd<-tix$Y<-0
for (i in unique(tix$LabelID)) {
	w<-which(tix$LabelID==i)
	tix$YEnd[w]<-tix$Y[w]<-seq(0,length(w)*0.04,length.out=length(w))+0.04
}
tix$Annotation<-''
tix$Association<-gmeans$Association[match(tix$LabelID,gmeans$LabelID)]
p+geom_signif(data=tix,mapping=aes(xmin=tix$X,xmax=tix$XEnd,annotations=tix$Annotation,
	y_position=tix$Y,tip_length=tix$tip_length),manual=T,inherit.aes=F)

```

## Hypothesis-driven analysis of ROI volumes {-}

We first assessed both regional volume at initial MRI and longitudinal volume change in each group relative to matched controls. This analysis indicated regions that displayed initial atrophy with no longitudinal progression (Table 2, blue cells), both initial atrophy and longitudinal progression (green cells), and progression in the absence of inital atrophy (red cells). In cross-sectional analysis of participants' initial MRI scans, all ROIs exhibited a main effect of group, independent of age and MMSE. The aAD patients exhibited initial atrophy relative to controls in all six MTL regions but in none of the other regions tested (Table 2). LvPPA patients exhibited atrophy relative to controls in left anterior insula and hippocampus as well as left parahippocampal, superior temporal, middle temporal, and middle frontal gyri. PCA patients, in turn, had lower volumes than controls in right hippocampus, angular gyrus, precuneus, superior parietal lobule, and middle temporal gyrus. FvAD patients had significant atrophy in left anterior insula and middle frontal gyrus; right angular gyrus; and bilateral middle temporal gyri.

In longitudinal models, all ROIs tested also exhibited a significant group x time interaction (Table 2). Post-hoc contrasts indicated areas where each patient group had significantly more rapid atrophy than controls. The aAD group differed from controls in all regions tested except for left entorhinal cortex, left anterior insula, and right angular gyrus. LvPPA patients similarly exhibited significant change over time in all regions except left anterior insula; and PCA patients differed from controls in all regions except left entorhinal cortex, parahipppocampal gyrus, and anterior insula. FvAD patients were the only group to exhibit significant atrophy over time in left anterior insula; they also had more rapid atrophy than controls in bilateral entorhinal cortex and middle temporal gyri as well as left superior temporal gyrus.

Post-hoc comparisons between patient groups reflected established phenotype-specific patterns of atrophy (Figure 1). LvPPA patients had lower volumes in left middle and superior temporal gyri than aAD patients (all z>=2.5, p<0.05); conversely, they had larger volumes in bilateral entorhinal cortex, right hippocampus, and right parahippocampal gyrus than aAD patients. PCA patients had lower volumes than aAD patients in the right precuneus and superior parietal lobule (z=3.2, p<0.02 and z=5.1, p<0.001, respectively); additionally, they exhibited marginally greater volumes than aAD patients in left hippocampus (z=2.3, p<0.08) and entorhinal cortex (z=2.2, p<0.09) and marginally lower volumes in right middle temporal gyrus (z=2.4, p<0.06). FvAD patients had more severe atrophy than aAD patients in the right superior parietal lobule (z=2.6, p<0.04) and marginally lower volumes in left middle frontal (z=2.5, p<0.06) and right middle temporal gyri (z=2.2, p<0.09). Conversely, like the lvPPA group, fvAD patients had greater volumes than the aAD group in left entorhinal cortex (z=2.7, p<0.04); they also had marginally greater volumes than aAD patients in right entorhinal cortex (z=2.1, p<0.09) and left parahippocampal gyrus (z=2.3, p<0.08). Comparisons between naAD groups further corroborated the association of each naAD phenotype with focal neocortical disease patterns. LvPPA patients had more severe initial atrophy than both PCA and fvAD patients in left superior temporal gyrus; and more severe atrophy than PCA patients in left middle temporal gyrus and anterior insula (all z>=2.5, p<0.05). PCA patients had lower volumes than lvPPA patients in right angular gyrus, precuneus, superior parietal lobule, and middle temporal gyrus, reflecting a phenotypic difference in lateralization of disease (all z>=2.6, p<0.05). Finally, fvAD patients had more severe atrophy than PCA patients in left middle frontal gyrus (z=2.6, p<0.05) and relative to lvPPA patients in right middle temporal gyrus and angular gyrus (z=2.8 and z=2.7, respectively, p<0.05).

```{r roi_time_diffs, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE,fig.align='center',fig.width=7.5,fig.height=9,fig.path='./fig/',fig.cap='Patient group differences in the effect of time for a priori regions of interest. The plot displays regression coefficients for annualized change in regional grey matter volume in each group, expressed in z-score units relative to the healthy control reference group. Error bars represent the standard error of each regression coefficient. More negative values on the x-axis indicate more rapid atrophy over time. Black brackets indicate significant pairwise group differences (p<0.05 after false discovery rate correction).'}

dodge<-position_dodge(width=0.64)
limits<-aes(ymax=gtimes$Beta+gtimes$SE,ymin=gtimes$Beta-gtimes$SE)

p2<-ggplot(data=gtimes,aes(x=ObsLab,y=Beta,fill=Group))+
	geom_bar(stat='identity',position=dodge)+geom_errorbar(limits,width=0.3,
		position=dodge,colour='grey50')+coord_flip()+
	theme(axis.text.y=element_text(size=10),axis.text.x=element_text(size=10,
		face='bold'),axis.title.y=element_blank(),axis.title.x=
		element_text(size=10,face='bold'),axis.ticks.y=element_blank(),
		axis.ticks.x=element_blank(),legend.position='top',
		legend.title=element_blank(),plot.title=element_text(size=10,
		face='bold'))+
	labs(y='Change in volume over time')
gb2<-ggplot_build(p2)
tix2<-gb2$layout$panel_params[[1]]$y.major_source
tix2<-data.frame(Mid=rep(tix2,each=6),Inc1=rep(c(0.08,-0.08,-0.08,-0.24,-0.24,-0.24),
	length(tix2)),Inc2=rep(c(0.24,0.08,0.24,-0.08,0.08,0.24),length(tix2)))
tix2$X=tix2$Mid+tix2$Inc1
tix2$XEnd<-tix2$Mid+tix2$Inc2
tix2$tip_length<-0.005
tix2$LabelID<-unique(gtimes$LabelID)[rev(tix2$Mid)]
tix2$G1<-rep(c('fvAD','PCA','fvAD','aAD','aAD','fvAD'),length.out=nrow(tix2))
tix2$G2<-rep(c('PCA','lvPPA','lvPPA','lvPPA','PCA','aAD'),length.out=nrow(tix2))
tmp<-subset(posthoc,grepl('time',posthoc$Comp) & !grepl('Control',posthoc$Comp))
tix2$P<-tmp$P[match(paste(tix2$G1,tix2$G2,tix2$LabelID,sep='_'),paste(tmp$G1,
	tmp$G2,tmp$LabelID,sep='_'))]
tix2<-subset(tix2,tix2$P<0.05)
tix2$YEnd<-tix2$Y<-0
for (i in unique(tix2$LabelID)) {
	w<-which(tix2$LabelID==i)
	tix2$YEnd[w]<-tix2$Y[w]<-seq(0,length(w)*0.01,length.out=length(w))+0.01
}
tix2$Annotation<-''
tix2$Association<-gtimes$Association[match(tix2$LabelID,gtimes$LabelID)]
p2+geom_signif(data=tix2,mapping=aes(xmin=tix2$X,xmax=tix2$XEnd,
	annotations=tix2$Annotation,y_position=tix2$Y,tip_length=tix2$tip_length),
	manual=T,inherit.aes=F)

```
Pairwise contrasts of group x time interaction terms (Figure 2) revealed differences in rates of longitudinal atrophy. We had hypothesized that each phenotype would exhibit faster atrophy in its assocaited neocortical ROIs than other patient groups; this hypothesis was upheld in left superior temporal gyrus, where lvPPA patients had more rapid atrophy than aAD patients (z=-2.6, p<0.03). In left anterior insula, fvAD patients had a faster rate of atrophy than both PCA (z=-2.7, p<0.03) and lvPPA (z=-2.5, p<0.05) patients. Longitudinal rates of atrophy did not differ between lvPPA and PCA in any of the regions investigated. Conversely, fvAD patients had slower atrophy than PCA in right precuneus (z=2.6, p<0.04). Contrary to hypotheses, naAD patients exhibited more gradual atrophy than aAD patients in several areas. PCA and lvPPA groups both had more gradual atrophy than aAD patients in right middle temporal gyrus (z=2.8, p<0.02 and z=3.0, p<0.02, respectively), and fvAD patients had slower atrophy relative to aAD in the right superior parietal lobule (z=2.3, p<0.05) and right precuneus (z=3.3, p<0.005). Additionally, the lvPPA and fvAD groups had slower atrophy than aAD patients in right supramarginal gyrus (all z>=3.2, p<0.01). MTL atrophy rates did not differ between naAD and aAD, except in left entorhinal cortex, where fvAD patients had more rapid atrophy than aAD patients (z=-2.4, p<0.05).

```{r age_effects, eval=TRUE, echo=FALSE, include=FALSE, results='asis'}

mtllabs<-c(47:48,116:117,170:171)
i<-mtllabs[1]
labid<-paste('lab',sprintf('%03d',i),sep='')
fm<-as.formula(paste(labid,'~Group*base_ageatMRI+base_MMSETotal+base_MMSETimeDiff',sep=''))
m<-lm(fm,data=subset(df,!duplicated(df$INDDID)))
nc<-length(coef(m))

gnames<-unique(df$Group)
tmp<-expand.grid(1:5,1:5)
tmp<-subset(tmp,!(tmp$Var1==tmp$Var2 | tmp$Var1>tmp$Var2))
gind<-tmp[rep(1:nrow(tmp),2),]
gind$G1<-gnames[tmp$Var1+1]
gind$G2<-gnames[tmp$Var2+1]
tmp$Var1[tmp$Var1==1]<-0
tmp<-tmp[order(tmp$Var1,tmp$Var2),]
tmp2<-tmp+7
tmp2$Var1[tmp2$Var1==7]<-0
tmp2<-tmp2[order(tmp2$Var1,tmp2$Var2),]
tmp<-rbind(tmp,tmp2)

kba<-matrix(0,nrow=length(gnames)*(length(gnames)-1),ncol=nc)
for (i in 1:nrow(kba)) { 
	kba[i,tmp$Var1[i]]<-(-1)
	kba[i,tmp$Var2[i]]<-1
}
kba<-data.matrix(kba)
rownames(kba)<-paste(rep(c('mean','age'),each=10),'_',gind$G2,'-',
	gind$G1,sep='')

pba<-matrix(NA,ncol=length(mtllabs),nrow=5)
phocba<-data.frame()
tba<-data.frame()
for (ind in 1:length(mtllabs)) {
	i<-mtllabs[ind]
	labid<-paste('lab',sprintf('%03d',i),sep='')
	labName<-roi$Label.Name[match(i,roi$LabelID)]

	# Linear regression testing group x age interaction at baseline.
	fm<-as.formula(paste(labid,'~Group*base_ageatMRI+base_MMSETotal+base_MMSETimeDiff',sep=''))
	m<-lm(fm,data=subset(df,!duplicated(df$INDDID)))
	am<-anova(m)
	tt<-summary(m)$coefficients
	tmp2<-as.data.frame(tt)
	tmp2$Effect<-rownames(tmp2)
	tmp2$LabelID=i; tmp2$Label.Name=labName
	tba<-rbind(tba,tmp2)

	# Post-hoc contrasts.
	pba[,ind]<-am$"Pr(>F)"[1:nrow(pba)]
	comp <- glht(m, linfct=kba)
	scomp<-summary(comp,test=adjusted('none'))
	scomp$test$pvalues[1:10]<-p.adjust(scomp$test$pvalues[1:10],'fdr')
	scomp$test$pvalues[11:20]<-p.adjust(scomp$test$pvalues[11:20],'fdr')
	tmp<-data.frame(LabelID=i,Comp=names(scomp$test$pvalues),Z=scomp$test$tstat,
		P=scomp$test$pvalues)
	phocba<-rbind(phocba,tmp)

	# Create a scatterplot for each ROI, with separate regression lines for each group.
	ageplot<-ggplot(data=subset(df,!duplicated(df$INDDID)),
		aes(x=base_ageatMRI,y=df[!duplicated(df$INDDID),labid],color=Group2,
		shape=Group2))+
		guides(color='none',shape='none')+
		geom_point()+
		geom_smooth(method='lm')+
		labs(x='Age at initial MRI',y='Normalized volume',
			title=unique(roi$Label.Name[roi$LabelID==i]))
	ggsave(filename=paste('fig/base_age_',labid,'.png',sep=''),plot=ageplot)
}

pba<-t(pba)
pbaadj<-apply(pba,2,function(x) { p.adjust(x,method='fdr') })

#mtlnames<-c('lab047','lab048','lab116','lab117','lab170','lab171')
#mtldf<-reshape(df,direction='long',varying=list(mtlnames),v.names='Volume',
#	timevar='Label',times=mtlnames,
#	idvar=names(df)[!grepl('lab',names(df))],
#	new.row.names=1:(length(mtlnames)*nrow(df)),
#	drop=setdiff(names(df)[grepl('lab',names(df))],mtlnames))
#mtlmod<-lm(Volume~Group*base_ageatMRI+base_MMSETotal+base_MMSETimeDiff,
#	data=subset(mtldf,!duplicated(mtldf$INDDID)))

```

### Effects of global cognition and age {-}
Initial MMSE score (which was included as a measure of global cognitive impairment) was positively associated with GM volume in all cortical ROIs outside the MTL, including bilateral middle temporal gyri; left anterior insula, middle frontal gyrus, and superior temporal gyrus; and right supramarginal gyrus, angular gyrus, precuneus, and superior parietal lobule [all t(94)>=2.6, p<0.02]. In the MTL, MMSE score was associated only with left and right entorhinal cortex volume [t(94)=3.4, p<0.01 and t(94)=2.2, p<0.05, respectively]. In contrast, age at initial MRI was inversely associated with volume in all 6 MTL regions investigated, including left and right hippocampus [t(94)=-2.4, p<0.05 and t(94)=-2.9, p<0.02, respectively, after FDR correction], left and right parahippocampal gyri [t(94)=-4.1, p<0.001 and t(94)=-4.5, p<0.001], and left and right entorhinal cortex [both t(94)=-3.4, p<0.001]; no other ROIs displayed an effect of age. To determine whether this age effect differed by group, we performed secondary analyses on MTL volumes at the time of first scan using multiple regression models with predictors of group, age, and their interaction, covarying for MMSE score and the interval between MMSE and MRI. After FDR correction, no MTL regions showed a significant group x age interaction [all F(4,90)<2.8, p>0.1], suggesting that the association of increased age with MTL atrophy was similar across groups.
 
```{r young_old, eval=TRUE, echo=FALSE, include=FALSE, results='asis',autodep=TRUE}

i<-hyplabs[1]
fm3<-as.formula(paste('lab',sprintf('%03d',i),
	'~Group3*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI',sep=''))
m3<-lme(fixed=fm3, random=~1|Subj,data=df,na.action=na.exclude)
nc<-length(fixef(m3))
gnames<-sort(unique(df$Group3))
kmat3<-matrix(0,nrow=length(gnames)*(length(gnames)-1),ncol=nc)
kmat3[6:9,2]<-
	kmat3[10:12,3]<-
	kmat3[13:14,4]<-
	kmat3[15,5]<-
	kmat3[21:24,11]<-
	kmat3[25:27,12]<-
	kmat3[28:29,13]<-
	kmat3[30,14]<-(-1)
kmat3[1,2]<-kmat3[2,3]<-kmat3[3,4]<-kmat3[4,5]<-kmat3[5,6]<-
	kmat3[6,3]<-kmat3[7,4]<-kmat3[8,5]<-kmat3[9,6]<-
	kmat3[10,4]<-kmat3[11,5]<-kmat3[12,6]<-
	kmat3[13,5]<-kmat3[14:15,6]<-
	kmat3[16,11]<-kmat3[17,12]<-kmat3[18,13]<-kmat3[19,14]<-kmat3[20,15]<-
	kmat3[21,12]<-kmat3[22,13]<-kmat3[23,14]<-kmat3[24,15]<-
	kmat3[25,13]<-kmat3[26,14]<-kmat3[27,15]<-
	kmat3[28,14]<-kmat3[29:30,15]<-1
kmat3<-data.matrix(kmat3)
rownames(kmat3)<-paste(rep(c('mean','time'),each=15),'_',rep(gnames[c(2:6,3:6,4:6,5:6,6)],2),
	'-',rep(gnames[c(rep(1,5),rep(2,4),rep(3,3),rep(4,2),5)],2),sep='')

pyo<-data.frame()
tyo<-data.frame()
page3<-page<-matrix(NA,ncol=length(hyplabs),nrow=7)

for (ind in 1:length(hyplabs)) {
	i<-hyplabs[ind]
	fm<-as.formula(paste('lab',sprintf('%03d',i),
		'~Group+AgeBin*time_diff+base_MMSETotal+base_MMSETimeDiff',sep=''))
	fm3<-as.formula(paste('lab',sprintf('%03d',i),
		'~Group3*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI',sep=''))
	m1<-lme(fixed=fm, random=~1|Subj,data=df,na.action=na.exclude)
	m3<-lme(fixed=fm3, random=~1|Subj,data=df,na.action=na.exclude)
	am1<-anova(m1)
	am3<-anova(m3)
	page[,ind]<-am1$"p-value"
	page3[,ind]<-am3$"p-value"
	comp <- glht(m3, linfct=kmat3)
	scomp<-summary(comp,test=adjusted('none'))
	scomp$test$pvalues[1:15]<-p.adjust(scomp$test$pvalues[1:15],'fdr')
	scomp$test$pvalues[16:30]<-p.adjust(scomp$test$pvalues[16:30],'fdr')
	tmp<-data.frame(LabelID=i,Comp=names(scomp$test$pvalues),Z=scomp$test$tstat,
		P=scomp$test$pvalues)
	pyo<-rbind(pyo,tmp)

	labName<-roi$Label.Name[match(i,roi$LabelID)]

	tt<-summary(m3)$tTable
	tmp2<-as.data.frame(tt)
	tmp2$Effect<-rownames(tmp2)
	tmp2$LabelID=i; tmp2$Label.Name=labName
	tyo<-rbind(tyo,tmp2)

	cat(labName,'\n')
	print(am1)
	print(summary(m1))
	cat(labName,'\n')
	print(am3)
	print(summary(m3))
	print(scomp)
}

page<-t(page)
page3<-t(page3)
pageadj<-apply(page,2,function(x) { p.adjust(x,method='fdr') })
pageadj3<-apply(page3,2,function(x) { p.adjust(x,method='fdr') })

pyo$Comp<-as.character(pyo$Comp)
pyo$G1<-NA;pyo$G2<-NA
for (i in 1:nrow(pyo)) {
	s<-unlist(strsplit(pyo$Comp[i],'-'))
	pyo$G1[i]<-unlist(strsplit(s[1],'_'))[3]
	pyo$G2[i]<-unlist(strsplit(s[2],'_'))[2]
}

pyo$Label.Name<-roi$Label.Name[match(pyo$LabelID,roi$LabelID)]
rownames(pyo)<-1:nrow(pyo)

posthoc$tag<-paste(posthoc$LabelID,posthoc$Comp,sep='_')
pyo$tag<-paste(pyo$LabelID,pyo$Comp,sep='_')

sig1<-setdiff(posthoc$tag[posthoc$P<0.05],pyo$tag[pyo$P<0.05])
sig2<-setdiff(pyo$tag[pyo$P<0.05],posthoc$tag[posthoc$P<0.05])

meanload<-pyo[pyo$P<0.05 & grepl('mean',pyo$Comp) & grepl('LOAD',pyo$Comp),c('Label.Name','Comp','Z','P')]
timeload<-pyo[pyo$P<0.05 & grepl('time',pyo$Comp) & grepl('LOAD',pyo$Comp),c('Label.Name','Comp','Z','P')]
meaneoad<-pyo[pyo$P<0.05 & grepl('mean',pyo$Comp) & grepl('EOAD',pyo$Comp),c('Label.Name','Comp','Z','P')]
timeeoad<-pyo[pyo$P<0.05 & grepl('time',pyo$Comp) & grepl('EOAD',pyo$Comp),c('Label.Name','Comp','Z','P')]

tmpsig1<-gsub('g3_aAD','g3a_EOAD',sig1)
tmpsig1<-c(tmpsig1,gsub('g3_aAD','g3b_LOAD',sig1))

notad<-setdiff(tmpsig1,sig2)
notad<-notad[!(grepl('LOAD',notad) | grepl('EOAD',notad))]
adrel<-setdiff(setdiff(tmpsig1,sig2),notad)
eoadrel<-adrel[grep('EOAD',adrel)]
loadrel<-adrel[grep('LOAD',adrel)]

```

### Effects of APOE genotype {-}

```{r lme_apoe, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

tabapoe<-table(tab1df$Group3[!tab1df$Group3 %in% 'g0_Control'],
	tab1df$APOE[!tab1df$Group3 %in% 'g0_Control'],useNA='ifany')
for (i in 1:nrow(tabapoe)) {
	s<-unlist(strsplit(rownames(tabapoe)[i],'_'))
	rownames(tabapoe)[i]<-s[2]
}

tabapoe2<-table(tab1df$Group[!tab1df$Group %in% 'g0_Control'],
	tab1df$APOE[!tab1df$Group %in% 'g0_Control'])
for (i in 1:nrow(tabapoe2)) {
	s<-unlist(strsplit(rownames(tabapoe2)[i],'_'))
	rownames(tabapoe2)[i]<-s[2]
}

#xtabapoe<-xtable(tabapoe,
#	caption='Distribution of APOE genotypes among aAD and naAD patients.',
#	align=rep('r',ncol(tabapoe)+1),digits=rep(0,ncol(tabapoe)+1))
#print(xtabapoe,comment=FALSE,caption.placement='top',floating=T,include.rownames=T,
#	sanitize.text.function=identity)

te4<-table(df$Group[!df$Group %in% 'g0_Control'],
	df$e4[!df$Group %in% 'g0_Control'])
te2<-table(df$Group[!df$Group %in% 'g0_Control'],
	df$e2[!df$Group %in% 'g0_Control'])
x2e4<-chisq.test(te4,simulate.p.value=T,B=10000)
x2e2<-chisq.test(te2,simulate.p.value=T,B=10000)

#m_apoe_onset<-lm(AgeatOnset~Group3+e4+e2,data=tab1df)
m_apoe_onset<-lm(AgeatOnset~Group+factor(e4),data=tab1df)

i<-hyplabs[1]
fm<-as.formula(paste('lab',sprintf('%03d',i),
	'~Group*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI+e4',sep=''))
m<-lme(fixed=fm, random=~1|Subj,data=df,na.action=na.exclude)
nc<-length(fixef(m))
gnames<-sort(unique(df$Group))

tmp<-expand.grid(1:4,1:4)
tmp<-subset(tmp,!(tmp$Var1==tmp$Var2 | tmp$Var1>tmp$Var2))
gind<-tmp[rep(1:nrow(tmp),2),]
gind$G1<-gnames[tmp$Var1+1]
gind$G2<-gnames[tmp$Var2+1]
tmp$Var1[tmp$Var1==1]<-0
tmp<-tmp[order(tmp$Var1,tmp$Var2),]
tmp2<-tmp+8
tmp2$Var1[tmp2$Var1==8]<-0
tmp2$Var2[tmp2$Var1==8]<-0
tmp2<-tmp2[order(tmp2$Var1,tmp2$Var2),]
tmp<-rbind(tmp,tmp2)

ke4<-matrix(0,nrow=length(gnames[-1])*(length(gnames[-1])-1),ncol=nc)
for (i in 1:nrow(ke4)) { 
	ke4[i,tmp$Var1[i]]<-(-1)
	ke4[i,tmp$Var2[i]]<-1
}
ke4<-data.matrix(ke4)
rownames(ke4)<-paste(rep(c('mean','time'),each=6),'_',gind$G2,'-',
	gind$G1,sep='')

phoce4<-data.frame()
pmate4<-matrix(NA,ncol=length(hyplabs),nrow=nrow(anova(m)))
ttabe4<-data.frame()

for (ind in 1:length(hyplabs)) {
	i<-hyplabs[ind]
	fm<-as.formula(paste('lab',sprintf('%03d',i),
		'~Group*time_diff+base_MMSETotal+base_MMSETimeDiff+base_ageatMRI+e4',sep=''))
	m<-lme(fixed=fm, random=~1|Subj,data=df,na.action=na.exclude)
	am<-anova(m)
	pmate4[,ind]<-am$"p-value"
	comp <- glht(m, linfct=ke4)
	scomp<-summary(comp,test=adjusted('none'))
	scomp$test$pvalues[1:6]<-p.adjust(scomp$test$pvalues[1:6],'fdr')
	scomp$test$pvalues[7:12]<-p.adjust(scomp$test$pvalues[7:12],'fdr')
	tmp<-data.frame(LabelID=i,Comp=names(scomp$test$pvalues),Z=scomp$test$tstat,
		P=scomp$test$pvalues)
	phoce4<-rbind(phoce4,tmp)

	labName<-roi$Label.Name[match(i,roi$LabelID)]
	tt<-summary(m)$tTable
	tmp2<-as.data.frame(tt)
	tmp2$Effect<-rownames(tmp2)
	tmp2$LabelID=i; tmp2$Label.Name=labName
	ttabe4<-rbind(ttabe4,tmp2)
}

pmate4<-t(pmate4)
padje4<-apply(pmate4,2,function(x) { p.adjust(x,method='fdr') })

phoce4$Comp<-as.character(phoce4$Comp)
phoce4$G1<-NA;phoce4$G2<-NA
for (i in 1:nrow(phoce4)) {
	s<-unlist(strsplit(phoce4$Comp[i],'-'))
	phoce4$G1[i]<-unlist(strsplit(s[1],'_'))[3]
	phoce4$G2[i]<-unlist(strsplit(s[2],'_'))[2]
}

phoce4$Label.Name<-roi$Label.Name[match(phoce4$LabelID,roi$LabelID)]
rownames(phoce4)<-1:nrow(phoce4)

```
We additionally assessed the distribution of APOE genotypes among aAD and naAD patients. Genotyping data were unavailable for 1 lvPPA, 1 fvAD, and 2 aAD patients. One aAD and 2 PCA patients each carried one copy of the $\epsilon2$ allele, which is associated with lower risk for Alzheimer's disease [@corder_protective_1994]. The proportions of lvPPA, PCA, fvAD, and aAD patients carrying 1--2 copies of the APOE $\epsilon4$ allele were 37.5%, 38.4%, 66.7%, and 84.6%, respectively. The frequency of individuals carrying 0, 1, or 2 copies of the $\epsilon4$ allele significantly differed across patient groups [$\chi^2$(6)=38.7, p<0.001]. These frequencies significantly differed between all phenotype pairs in post-hoc comparisons (all $\chi^2$(2)>=7.1, p<0.03). Because APOE genotypes were unavailable for control participants, we recomputed LMEs for a priori ROIs using lvPPA (the largest group) as the reference group and number of $\epsilon4$ alleles as a covariate. No ROIs exhibited an association with number of $\epsilon4$ alleles independent of group.

### Longitudinal associations between neuropsychological performance and grey matter volume {-}
```{r long_npsych, eval=TRUE, echo=FALSE, include=FALSE, results='asis',autodep=TRUE}

pvlt<-read.csv('INQuery_Longitudinal_PVLT.csv',stringsAsFactors=F)
pvlt$TestDate<-as.Date(pvlt$TestDate,format='%m/%d/%Y')
pvlt<-subset(pvlt,pvlt$INDDID %in% pvlt$INDDID[!is.na(pvlt$TestDate)])
pvlt<-pvlt[order(pvlt$INDDID,pvlt$TestDate),]
pvlt$Recognition<-pvlt$PHLRecognitionDiscrimin
pvlt$Source<-'PVLT'

pbac<-read.csv('INQuery_Longitudinal_PBAC.csv',stringsAsFactors=F)
pbac$TestDate<-as.Date(pbac$TestDate,format='%m/%d/%Y')
pbac<-subset(pbac,pbac$INDDID %in% pbac$INDDID[!is.na(pbac$TestDate)])
pbac<-pbac[order(pbac$INDDID,pbac$TestDate),]
pbac$Recognition<-pbac$VerbalMemoryRecognition/3
pbac$Source<-'PBAC'

xn<-intersect(names(pbac),names(pvlt))
recog<-rbind(pvlt[,xn],pbac[,xn])
recog<-recog[order(recog$INDDID,recog$TestDate),]
recog$Time<-0
for (i in 2:nrow(recog)) {
	if (recog$INDDID[i]==recog$INDDID[i-1]) {
		recog$Time[i]<-as.numeric(recog$TestDate[i]-recog$TestDate[i-1])/365.25
	}
}
out<-cleaner(recog$Time,outCheck=T,outThresh=3)
wout<-which(out)
recog<-recog[-wout,]
recog<-subset(recog,recog$INDDID %in% recog$INDDID[duplicated(recog$INDDID)])
recog$DOB<-df$DOB[match(recog$INDDID,df$INDDID)]
recog$AgeatTest<-as.numeric(recog$TestDate-recog$DOB)/365.25
recog$BaseAge<-recog$AgeatTest[match(recog$INDDID,recog$INDDID)]
recog$BaseRecognition<-recog$Recognition[match(recog$INDDID,recog$INDDID)]
recog$Group<-factor(as.character(recog$Group),levels=c('g1_lvPPA','g2_PCA',
	'g3_aAD','g4_fvAD','g0_Control'))
	
pbac$Time<-0
for (i in 2:nrow(pbac)) {
	if (pbac$INDDID[i]==pbac$INDDID[i-1]) {
		pbac$Time[i]<-as.numeric(pbac$TestDate[i]-pbac$TestDate[i-1])/365.25
	}
}
out<-cleaner(pbac$Time,outCheck=T,outThresh=3)
wout<-which(out)
pbac<-pbac[-wout,]
pbac<-subset(pbac,pbac$INDDID %in% pbac$INDDID[duplicated(pbac$INDDID)])
pbac$DOB<-df$DOB[match(pbac$INDDID,df$INDDID)]
pbac$AgeatTest<-as.numeric(pbac$TestDate-pbac$DOB)/365.25
pbac$BaseAge<-pbac$AgeatTest[match(pbac$INDDID,pbac$INDDID)]
pbac$Group<-factor(as.character(pbac$Group),levels=c('g1_lvPPA','g2_PCA',
	'g3_aAD','g4_fvAD','g0_Control'))
pbac$JOLO<-cleaner(pbac$JOLO)
pbac$ReyCopy<-cleaner(pbac$ReyCopy)

flet<-read.csv('INQuery_Longitudinal_FLetters.csv',stringsAsFactors=F)
flet$TestDate<-as.Date(flet$TestDate,format='%m/%d/%Y')
flet<-subset(flet,flet$INDDID %in% flet$INDDID[!is.na(flet$TestDate)])
flet<-flet[order(flet$INDDID,flet$TestDate),]
flet$Time<-0
for (i in 2:nrow(flet)) {
	if (flet$INDDID[i]==flet$INDDID[i-1]) {
		flet$Time[i]<-as.numeric(flet$TestDate[i]-flet$TestDate[i-1])/365.25
	}
}
out<-cleaner(flet$Time,outCheck=T,outThresh=3)
wout<-which(out)
flet<-flet[-wout,]
flet<-subset(flet,flet$INDDID %in% flet$INDDID[duplicated(flet$INDDID)])
flet$DOB<-df$DOB[match(flet$INDDID,df$INDDID)]
flet$AgeatTest<-as.numeric(flet$TestDate-flet$DOB)/365.25
flet$BaseAge<-flet$AgeatTest[match(flet$INDDID,flet$INDDID)]
flet$FLetterFluency<-cleaner(flet$FLetterFluency)
flet$Group<-factor(as.character(flet$Group),levels=c('g1_lvPPA','g2_PCA',
	'g3_aAD','g4_fvAD','g0_Control'))

digsp<-read.csv('INQuery_Longitudinal_DigitSpan.csv',stringsAsFactors=F)
digsp$Group<-df$Group[match(digsp$INDDID,df$INDDID)]
digsp$TestDate<-as.Date(digsp$TestDate,format='%m/%d/%Y')
digsp<-subset(digsp,digsp$INDDID %in% digsp$INDDID[!is.na(digsp$TestDate)])
digsp<-digsp[order(digsp$INDDID,digsp$TestDate),]
digsp$ForwardSpan<-cleaner(digsp$ForwardSpan)
digsp$ReverseSpan<-cleaner(digsp$ReverseSpan)
digsp$Time<-0
for (i in 2:nrow(digsp)) {
	if (digsp$INDDID[i]==digsp$INDDID[i-1]) {
		digsp$Time[i]<-as.numeric(digsp$TestDate[i]-digsp$TestDate[i-1])/365.25
	}
}
out<-cleaner(digsp$Time,outCheck=T,outThresh=3)
wout<-which(out)
digsp<-digsp[-wout,]
digsp<-subset(digsp,digsp$INDDID %in% digsp$INDDID[duplicated(digsp$INDDID)])
digsp$DOB<-df$DOB[match(digsp$INDDID,df$INDDID)]
digsp$AgeatTest<-as.numeric(digsp$TestDate-digsp$DOB)/365.25
digsp$BaseAge<-digsp$AgeatTest[match(digsp$INDDID,digsp$INDDID)]
digsp$Group<-factor(as.character(digsp$Group),levels=c('g1_lvPPA','g2_PCA',
	'g3_aAD','g4_fvAD','g0_Control'))

```

```{r npsych_mri, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

# Find closest MRI observation for each neuropsych observation.
# Omit duplicate MRI sessions, so that there's a one-to-one mapping.

labnum<-paste('lab',sprintf('%03d',hyplabs),sep='')
addcols<-c(labnum,'MRIDate','Interval','base_ageatMRI','base_MMSETotal',
	'base_MMSETimeDiff','MMSETotal','CxVol')
recog[,addcols]<-NA; recog$MRIDate<-as.Date(recog$MRIDate,format='%Y-%m-%d')
pbac[,addcols]<-NA; pbac$MRIDate<-as.Date(pbac$MRIDate,format='%Y-%m-%d')
flet[,addcols]<-NA; flet$MRIDate<-as.Date(flet$MRIDate,format='%Y-%m-%d')
fsp<-rsp<-digsp
fsp[,addcols]<-NA; fsp$MRIDate<-as.Date(fsp$MRIDate,format='%Y-%m-%d')
rsp[,addcols]<-NA; rsp$MRIDate<-as.Date(rsp$MRIDate,format='%Y-%m-%d')

for (i in 1:nrow(recog)) {
	tmp<-subset(df,df$INDDID==recog$INDDID[i])
	tmp$Interval<-as.numeric(tmp$MRIDate-recog$TestDate[i])/365.25
	tmp<-subset(tmp,abs(tmp$Interval)<=1)
	if (nrow(tmp)>0) {
		wmin<-which.min(abs(tmp$Interval))
		recog[i,addcols]<-tmp[wmin,addcols]
	}
}
recog_mri<-recog[!duplicated(recog[,c('INDDID','MRIDate')]),]
recog_mri<-subset(recog_mri,!(is.na(recog_mri$MRIDate) | 
	is.na(recog_mri$Recognition)))
recog_mri<-subset(recog_mri,!recog_mri$Group %in% 'g0_Control')
recog_mri<-subset(recog_mri,recog_mri$INDDID %in% 
	recog_mri$INDDID[duplicated(recog_mri$INDDID)])

for (i in 1:nrow(pbac)) {
	tmp<-subset(df,df$INDDID==pbac$INDDID[i])
	tmp$Interval<-as.numeric(tmp$MRIDate-pbac$TestDate[i])/365.25
	tmp<-subset(tmp,abs(tmp$Interval)<=1)
	if (nrow(tmp)>0) {
		wmin<-which.min(abs(tmp$Interval))
		pbac[i,addcols]<-tmp[wmin,addcols]
	}	
}
pbac_mri<-pbac[!duplicated(pbac[,c('INDDID','MRIDate')]),]
pbac_mri<-subset(pbac_mri,!is.na(pbac_mri$MRIDate))
pbac_mri<-subset(pbac_mri,!pbac_mri$Group %in% 'g0_Control')
pbac_mri<-subset(pbac_mri,pbac_mri$INDDID %in% 
	pbac_mri$INDDID[duplicated(pbac_mri$INDDID)])

for (i in 1:nrow(flet)) {
	tmp<-subset(df,df$INDDID==flet$INDDID[i])
	tmp$Interval<-as.numeric(tmp$MRIDate-flet$TestDate[i])/365.25
	tmp<-subset(tmp,abs(tmp$Interval)<=1)
	if (nrow(tmp)>0) {
		wmin<-which.min(abs(tmp$Interval))
		flet[i,addcols]<-tmp[wmin,addcols]
	}	
}
flet_mri<-flet[!duplicated(flet[,c('INDDID','MRIDate')]),]
flet_mri<-subset(flet_mri,!(is.na(flet_mri$MRIDate) |
	is.na(flet_mri$FLetterFluency)))
flet_mri<-subset(flet_mri,!flet_mri$Group %in% 'g0_Control')
flet_mri<-subset(flet_mri,flet_mri$INDDID %in% 
	flet_mri$INDDID[duplicated(flet_mri$INDDID)])

for (i in 1:nrow(fsp)) {
	tmp<-subset(df,df$INDDID==fsp$INDDID[i])
	tmp$Interval<-as.numeric(tmp$MRIDate-fsp$TestDate[i])/365.25
	tmp<-subset(tmp,abs(tmp$Interval)<=1)
	if (nrow(tmp)>0) {
		wmin<-which.min(abs(tmp$Interval))
		fsp[i,addcols]<-tmp[wmin,addcols]
	}	
}
fsp_mri<-fsp[!duplicated(fsp[,c('INDDID','MRIDate')]),]
fsp_mri<-subset(fsp_mri,!(is.na(fsp_mri$MRIDate) |
	is.na(fsp_mri$ForwardSpan)))
fsp_mri<-subset(fsp_mri,!fsp_mri$Group %in% 'g0_Control')
fsp_mri<-subset(fsp_mri,fsp_mri$INDDID %in% 
	fsp_mri$INDDID[duplicated(fsp_mri$INDDID)])

for (i in 1:nrow(rsp)) {
	tmp<-subset(df,df$INDDID==rsp$INDDID[i])
	tmp$Interval<-as.numeric(tmp$MRIDate-rsp$TestDate[i])/365.25
	tmp<-subset(tmp,abs(tmp$Interval)<=1)
	if (nrow(tmp)>0) {
		wmin<-which.min(abs(tmp$Interval))
		rsp[i,addcols]<-tmp[wmin,addcols]
	}	
}
rsp_mri<-rsp[!duplicated(rsp[,c('INDDID','MRIDate')]),]
rsp_mri<-subset(rsp_mri,!(is.na(rsp_mri$MRIDate) |
	is.na(rsp_mri$ForwardSpan)))
rsp_mri<-subset(rsp_mri,!rsp_mri$Group %in% 'g0_Control')
rsp_mri<-subset(rsp_mri,rsp_mri$INDDID %in% 
	rsp_mri$INDDID[duplicated(rsp_mri$INDDID)])

# Create baseline volume and volume change variables for each label.
recog_mri$BaseLab047<-recog_mri$lab047[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab047<-recog_mri$lab047-recog_mri$BaseLab047
recog_mri$BaseLab048<-recog_mri$lab048[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab048<-recog_mri$lab048-recog_mri$BaseLab048
recog_mri$BaseLab116<-recog_mri$lab116[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab116<-recog_mri$lab116-recog_mri$BaseLab116
recog_mri$BaseLab117<-recog_mri$lab117[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab117<-recog_mri$lab117-recog_mri$BaseLab117
recog_mri$BaseLab170<-recog_mri$lab170[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab170<-recog_mri$lab170-recog_mri$BaseLab170
recog_mri$BaseLab171<-recog_mri$lab171[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffLab171<-recog_mri$lab171-recog_mri$BaseLab171
recog_mri$BaseCxVol<-recog_mri$CxVol[match(recog_mri$INDDID,recog_mri$INDDID)]
recog_mri$DiffCxVol<-recog_mri$CxVol-recog_mri$BaseCxVol

flet_mri$BaseLab155<-flet_mri$lab155[match(flet_mri$INDDID,flet_mri$INDDID)]
flet_mri$DiffLab155<-flet_mri$lab155-flet_mri$BaseLab155
flet_mri$BaseLab201<-flet_mri$lab201[match(flet_mri$INDDID,flet_mri$INDDID)]
flet_mri$DiffLab201<-flet_mri$lab201-flet_mri$BaseLab201
flet_mri$BaseCxVol<-flet_mri$CxVol[match(flet_mri$INDDID,flet_mri$INDDID)]
flet_mri$DiffCxVol<-flet_mri$CxVol-flet_mri$BaseCxVol

fsp_mri$BaseLab155<-fsp_mri$lab155[match(fsp_mri$INDDID,fsp_mri$INDDID)]
fsp_mri$DiffLab155<-fsp_mri$lab155-fsp_mri$BaseLab155
fsp_mri$BaseLab201<-fsp_mri$lab201[match(fsp_mri$INDDID,fsp_mri$INDDID)]
fsp_mri$DiffLab201<-fsp_mri$lab201-fsp_mri$BaseLab201
fsp_mri$BaseCxVol<-fsp_mri$CxVol[match(fsp_mri$INDDID,fsp_mri$INDDID)]
fsp_mri$DiffCxVol<-fsp_mri$CxVol-fsp_mri$BaseCxVol

rsp_mri$BaseLab103<-rsp_mri$lab103[match(rsp_mri$INDDID,rsp_mri$INDDID)]
rsp_mri$DiffLab103<-rsp_mri$lab103-rsp_mri$BaseLab103
rsp_mri$BaseLab143<-rsp_mri$lab143[match(rsp_mri$INDDID,rsp_mri$INDDID)]
rsp_mri$DiffLab143<-rsp_mri$lab143-rsp_mri$BaseLab143
rsp_mri$BaseLab154<-rsp_mri$lab154[match(rsp_mri$INDDID,rsp_mri$INDDID)]
rsp_mri$DiffLab154<-rsp_mri$lab154-rsp_mri$BaseLab154
rsp_mri$BaseCxVol<-rsp_mri$CxVol[match(rsp_mri$INDDID,rsp_mri$INDDID)]
rsp_mri$DiffCxVol<-rsp_mri$CxVol-rsp_mri$BaseCxVol

pbac_mri$BaseLab103<-pbac_mri$lab103[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab103<-pbac_mri$lab103-pbac_mri$BaseLab103
pbac_mri$BaseLab106<-pbac_mri$lab106[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab106<-pbac_mri$lab106-pbac_mri$BaseLab106
pbac_mri$BaseLab154<-pbac_mri$lab154[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$BaseLab143<-pbac_mri$lab143[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab143<-pbac_mri$lab143-pbac_mri$BaseLab143
pbac_mri$DiffLab154<-pbac_mri$lab154-pbac_mri$BaseLab154
pbac_mri$BaseLab155<-pbac_mri$lab155[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab155<-pbac_mri$lab155-pbac_mri$BaseLab155
pbac_mri$BaseLab168<-pbac_mri$lab168[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab168<-pbac_mri$lab168-pbac_mri$BaseLab168
pbac_mri$BaseLab194<-pbac_mri$lab194[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab194<-pbac_mri$lab194-pbac_mri$BaseLab194
pbac_mri$BaseLab198<-pbac_mri$lab198[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab198<-pbac_mri$lab198-pbac_mri$BaseLab198
pbac_mri$BaseLab201<-pbac_mri$lab201[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffLab201<-pbac_mri$lab201-pbac_mri$BaseLab201
pbac_mri$BaseCxVol<-pbac_mri$CxVol[match(pbac_mri$INDDID,pbac_mri$INDDID)]
pbac_mri$DiffCxVol<-pbac_mri$CxVol-pbac_mri$BaseCxVol

# Models testing association of longitudinal recognition scores with 
# MTL ROI volumes.
fitsumm<-data.frame()
recog_lab047<-lme(Recognition~BaseLab047+DiffLab047+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_lab048<-lme(Recognition~BaseLab048+DiffLab048+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_lab116<-lme(Recognition~BaseLab116+DiffLab116+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_lab117<-lme(Recognition~BaseLab117+DiffLab117+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_lab170<-lme(Recognition~BaseLab170+DiffLab170+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_lab171<-lme(Recognition~BaseLab171+DiffLab171+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)
recog_cxvol<-lme(Recognition~BaseCxVol+DiffCxVol+Source,random=~1|INDDID,
	data=recog_mri,na.action=na.exclude)

# Add fixed effects tables to aggregate data frame of results.
tmp<-as.data.frame(summary(recog_lab047)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-47
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_lab048)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-48
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_lab116)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-116
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_lab117)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-117
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_lab170)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-170
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_lab171)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-171
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(recog_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Memory';tmp$Task<-'Recognition';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

# Models testing association of longitudinal language scores with 
# left temporal ROI volumes.

tmp<-subset(pbac_mri,!is.na(pbac_mri$Speech))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
speech_lab155<-lme(Speech~BaseLab155+DiffLab155,random=~1|INDDID,
	data=tmp,na.action=na.exclude)
speech_lab201<-lme(Speech~BaseLab201+DiffLab201,random=~1|INDDID,
	data=tmp,na.action=na.exclude)
speech_cxvol<-lme(Speech~BaseCxVol+DiffCxVol,random=~1|INDDID,
	data=tmp,na.action=na.exclude)

#tmp<-subset(pbac_mri,!is.na(pbac_mri$DigitsForward))
#tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
tmp<-subset(fsp_mri,!is.na(fsp_mri$ForwardSpan))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
fspan_lab155<-lme(ForwardSpan~BaseLab155+DiffLab155,random=~1|INDDID,
	data=tmp,na.action=na.exclude)
fspan_lab201<-lme(ForwardSpan~BaseLab201+DiffLab201,random=~1|INDDID,
	data=tmp,na.action=na.exclude)
fspan_cxvol<-lme(ForwardSpan~BaseCxVol+DiffCxVol,random=~1|INDDID,
	data=tmp,na.action=na.exclude)

flet_lab155<-lme(FLetterFluency~BaseLab155+DiffLab155,random=~1|INDDID,
	data=flet_mri,na.action=na.exclude)
flet_lab201<-lme(FLetterFluency~BaseLab201+DiffLab201,random=~1|INDDID,
	data=flet_mri,na.action=na.exclude)
flet_cxvol<-lme(FLetterFluency~BaseCxVol+DiffCxVol,random=~1|INDDID,
	data=flet_mri,na.action=na.exclude)

tmp<-as.data.frame(summary(speech_lab155)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'Speech';tmp$Label<-155
fitsumm<-rbind(fitsumm,tmp)
	
tmp<-as.data.frame(summary(speech_lab201)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'Speech';tmp$Label<-201
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(speech_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'Speech';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(flet_lab155)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'FLetterFluency';tmp$Label<-155
fitsumm<-rbind(fitsumm,tmp)
	
tmp<-as.data.frame(summary(flet_lab201)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'FLetterFluency';tmp$Label<-201
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(flet_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'FLetterFluency';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(fspan_lab155)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'ForwardSpan';tmp$Label<-155
fitsumm<-rbind(fitsumm,tmp)
	
tmp<-as.data.frame(summary(fspan_lab201)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'ForwardSpan';tmp$Label<-201
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(fspan_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Language';tmp$Task<-'ForwardSpan';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

# Associations between longitudinal visuospatial performance and PCA ROIs.
tmp<-subset(pbac_mri,!is.na(pbac_mri$ReyCopy))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
rey_lab106<-lme(ReyCopy~BaseLab106+DiffLab106,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
rey_lab154<-lme(ReyCopy~BaseLab154+DiffLab154,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
rey_lab168<-lme(ReyCopy~BaseLab168+DiffLab168,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
rey_lab194<-lme(ReyCopy~BaseLab194+DiffLab194,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
rey_lab198<-lme(ReyCopy~BaseLab198+DiffLab198,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
rey_cxvol<-lme(ReyCopy~BaseCxVol+DiffCxVol,
	random=~1|INDDID,data=tmp,na.action=na.exclude)

tmp<-subset(pbac_mri,!is.na(pbac_mri$JOLO))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
jolo_lab106<-lme(JOLO~BaseLab106+DiffLab106,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
jolo_lab154<-lme(JOLO~BaseLab154+DiffLab154,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
jolo_lab168<-lme(JOLO~BaseLab168+DiffLab168,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
jolo_lab194<-lme(JOLO~BaseLab194+DiffLab194,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
jolo_lab198<-lme(JOLO~BaseLab198+DiffLab198,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
jolo_cxvol<-lme(JOLO~BaseCxVol+DiffCxVol,
	random=~1|INDDID,data=tmp,na.action=na.exclude)

tmp<-as.data.frame(summary(rey_lab106)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-106
fitsumm<-rbind(fitsumm,tmp)
	
tmp<-as.data.frame(summary(rey_lab154)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-154
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(rey_lab168)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-168
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(rey_lab194)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-194
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(rey_lab198)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-198
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(rey_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'ReyCopy';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(jolo_lab106)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-106
fitsumm<-rbind(fitsumm,tmp)
	
tmp<-as.data.frame(summary(jolo_lab154)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-154
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(jolo_lab168)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-168
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(jolo_lab194)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-194
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(jolo_lab198)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-198
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(jolo_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Visuospatial';tmp$Task<-'JOLO';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

# Associations between longitudinal behavioral/executive performance and 
# fvAD ROIs.
tmp<-subset(pbac_mri,!is.na(pbac_mri$BehavioralScale))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
bhav_lab103<-lme(BehavioralScale~BaseLab103+DiffLab103,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bhav_lab143<-lme(BehavioralScale~BaseLab143+DiffLab143,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bhav_lab154<-lme(BehavioralScale~BaseLab154+DiffLab154,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bhav_cxvol<-lme(BehavioralScale~BaseCxVol+DiffCxVol,
	random=~1|INDDID,data=tmp,na.action=na.exclude)

tmp<-subset(pbac_mri,!is.na(pbac_mri$OralTrails))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
otrails_lab103<-lme(OralTrails~BaseLab103+DiffLab103,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
otrails_lab143<-lme(OralTrails~BaseLab143+DiffLab143,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
otrails_lab154<-lme(OralTrails~BaseLab154+DiffLab154,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
otrails_cxvol<-lme(OralTrails~BaseCxVol+DiffCxVol,
	random=~1|INDDID,data=tmp,na.action=na.exclude)

#tmp<-subset(pbac_mri,!is.na(pbac_mri$DigitsBackward))
tmp<-subset(rsp_mri,!is.na(rsp_mri$ReverseSpan))
tmp<-subset(tmp,tmp$INDDID %in% tmp$INDDID[duplicated(tmp$INDDID)])
bspan_lab103<-lme(ReverseSpan~BaseLab103+DiffLab103,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bspan_lab143<-lme(ReverseSpan~BaseLab143+DiffLab143,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bspan_lab154<-lme(ReverseSpan~BaseLab154+DiffLab154,
	random=~1|INDDID,data=tmp,na.action=na.exclude)
bspan_cxvol<-lme(ReverseSpan~BaseCxVol+DiffCxVol,
	random=~1|INDDID,data=tmp,na.action=na.exclude)

tmp<-as.data.frame(summary(bhav_lab103)$tTable,stringsAsFactors=F)
tmp$Domain<-'Behavioral';tmp$Task<-'BehavioralScale';tmp$Label<-103
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bhav_lab143)$tTable,stringsAsFactors=F)
tmp$Domain<-'Behavioral';tmp$Task<-'BehavioralScale';tmp$Label<-143
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bhav_lab154)$tTable,stringsAsFactors=F)
tmp$Domain<-'Behavioral';tmp$Task<-'BehavioralScale';tmp$Label<-154
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bhav_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Behavioral';tmp$Task<-'BehavioralScale';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(otrails_lab103)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'OralTrails';tmp$Label<-103
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(otrails_lab143)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'OralTrails';tmp$Label<-143
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(otrails_lab154)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'OralTrails';tmp$Label<-154
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(otrails_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'OralTrails';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bspan_lab103)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'ReverseSpan';tmp$Label<-103
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bspan_lab143)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'ReverseSpan';tmp$Label<-143
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bspan_lab154)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'ReverseSpan';tmp$Label<-154
fitsumm<-rbind(fitsumm,tmp)

tmp<-as.data.frame(summary(bspan_cxvol)$tTable,stringsAsFactors=F)
tmp$Domain<-'Executive';tmp$Task<-'ReverseSpan';tmp$Label<-'CxVol'
fitsumm<-rbind(fitsumm,tmp)

names(fitsumm)[4]<-'Tstat'
fitsumm$Task<-gsub('Recognition','Recognition memory',fitsumm$Task)
fitsumm$Task<-gsub('FLetterFluency','Letter fluency',fitsumm$Task)
fitsumm$Task<-gsub('ForwardSpan','Forward digit span',fitsumm$Task)
fitsumm$Task<-gsub('ReyCopy','Rey copy',fitsumm$Task)
fitsumm$Task<-gsub('JOLO','Judgment of line orientation',fitsumm$Task)
fitsumm$Task<-gsub('BehavioralScale','Social behavior',fitsumm$Task)
fitsumm$Task<-gsub('OralTrails','Oral Trails',fitsumm$Task)
#fitsumm$Task<-gsub('DigitsBackward','Digits backward',fitsumm$Task)
fitsumm$Task<-gsub('ReverseSpan','Reverse digit span',fitsumm$Task)
fitsumm$Task<-factor(fitsumm$Task,levels=c('Recognition memory','Speech',
	'Letter fluency','Forward digit span','Rey copy','Judgment of line orientation','Social behavior','Oral Trails',
	'Reverse digit span'))
fitsumm$Domain<-factor(fitsumm$Domain,levels=c('Memory','Language',
	'Visuospatial','Behavioral','Executive'))
fitsumm<-fitsumm[order(fitsumm$Task,fitsumm$Label),]
fitsumm$Control<-ifelse(fitsumm$Label=='CxVol',TRUE,FALSE)

# Which ROIs do better than the cortical average in predicting task performance?
sigeff<-data.frame()
for (task in unique(fitsumm$Task)) {
	cxtstat<-fitsumm$Tstat[fitsumm$Task %in% task & grepl('DiffCxVol',
		rownames(fitsumm))]
	siglab<-unique(fitsumm$Label[fitsumm$Task %in% task &
		grepl('DiffLab',rownames(fitsumm)) &
		abs(fitsumm$Tstat)>=abs(cxtstat) &
		fitsumm$"p-value"<0.05])
	tmp<-subset(fitsumm,fitsumm$Task %in% task & fitsumm$Label %in% siglab)
	sigeff<-rbind(sigeff,tmp)
}

fitsumm$Region<-anovatab2$Region[match(fitsumm$Label,anovatab2$LabelID)]
fitsumm$Region[fitsumm$Label=='CxVol']<-'Whole cortex'
fitsumm$P<-fitsumm$"p-value"

fitsumm$T<-paste('t(',fitsumm$DF,')=',sprintf('%.1f',fitsumm$Tstat),sep='')
npsytab<-fitsumm[grepl('Diff',rownames(fitsumm)),c('Domain','Task','Region',
	'T','P')]
npsytab$P<-p.adjust(npsytab$P,'fdr')
w1<-which(npsytab$P>=0.1)
npsytab$P[w1]<-ceiling(10*npsytab$P[w1])/10
w2<-which(npsytab$P<0.1 & npsytab$P>=0.01)
npsytab$P[w2]<-ceiling(100*npsytab$P[w2])/100
w3<-which(npsytab$P<0.01 & npsytab$P>=0.001)
npsytab$P[w3]<-ceiling(1000*npsytab$P[w3])/1000
w4<-which(npsytab$P<0.001)
npsytab$P[w4]<-sprintf('%.4f',ceiling(10000*npsytab$P[w4])/10000)
npsytab$P<-as.character(npsytab$P)
npsytab<-npsytab[order(npsytab$Domain,npsytab$Task,npsytab$Region),]
npsytab$Task<-as.character(npsytab$Task)
npsytab$Task[duplicated(npsytab$Task)]<-''
npsytab<-subset(npsytab,!npsytab$Region %in% 'Whole cortex')
xnpsytab<-xtable(npsytab[,-1],
	caption='Associations between neuropsychological performance and grey matter volume change in task-specific ROIs. P-values are corrected for multiple comparisons using the false discovery rate method; values<0.05 are considered statistically significant and shown in bold.',
	align='lllll',digits=0)
wsig<-which(as.numeric(xnpsytab$P)<=0.05)
xnpsytab$Region[wsig]<-paste('\\textbf{',xnpsytab$Region[wsig],'}',sep='')
xnpsytab$T[wsig]<-paste('\\textbf{',xnpsytab$T[wsig],'}',sep='')
xnpsytab$P[wsig]<-paste('\\textbf{',xnpsytab$P[wsig],'}',sep='')
print(xnpsytab,comment=FALSE,caption.placement='top',floating=T,
	include.rownames=F,sanitize.text.function=identity,
	sanitize.colnames.function=identity)

```

Associations between longitudinal neuropsychological performance and atrophy were evaluated in patients that had two or more assessments, each within one year of a structural MRI scan. For recognition memory, this yielded 109 observations from 45 patients, including 21 lvPPA, 9 PCA, 7 aAD, and 8 fvAD patients. For letter fluency, 99 observations were available from 18 lvPPA, 9 PCA, 6 aAD, and 8 fvAD patients. For forward and reverse digit span, 107 observations were available from 22 lvPPA, 10 PCA, 6 aAD, and 7 fvAD patients. A total of 75 PBAC observations, from which all other test measures were obtained, were available for 17 lvPPA, 6 PCA, 5 aAD, and 6 fvAD patients. In all cognitive domains except for social behavior, longitudinal cognition was directly associated with GM volume change in one or more associated brain areas, independent of volume at the time of initial MRI (Table 3). In the memory domain, volume loss in bilateral hippocampi and entorhinal cortex as well as left parahippocampal gyrus predicted declines in recognition discrimination. In the language domain, volume loss in left middle temporal gyrus was associated with decreases in letter fluency; and volume loss in both left middle and superior temporal gyri predicted declines in forward digit span. In the visuospatial domain, Rey figure copy performance over time was associated with volume loss in right angular, middle temporal, and supramarginal gyrus as well as right precuneus. Judgment of line orientations was associated with right precuneus volume. No significant associations were found for social behavior; however, left middle frontal and right middle temporal gyri predicted reverse digit span, a measure of working memory and executive function [@kramer_distinctive_2003].

## Exploratory analysis {-}
\begin{figure}[]
% Voxelwise cortical thickness versus controls.

{\centering \includegraphics[width=7in]{./fig/me_time.pdf} 

}

\caption[GM time effects]{Voxelwise differences in cortical thickness relative to matched controls. Image overlays are binarized t-statistic maps for simple contrasts of controls minus each patient group. Blue: simple effect of group (patients<controls) from cross-sectional analysis of participants' initial MRI scans; red: group x time interaction from longitudinal LME models, indicating where patients have more rapid cortical thinning than controls; green: overlap between group and group x time effects. All results were calculated from linear mixed effects (LME) models and thresholded at voxelwise p<0.001 with a minimum cluster volume of \SI{560}{\micro\litre}, corresponding to a corrected cluster-wise threshold of p<0.05. \label{gm_me_time}}

\end{figure}

```{r run_vox_base, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

# Run voxelwise linear models on cortical thickness maps from initial scans.
basevox<-read.table('~/bin/long_lme_cxthick_20180525.sh',skip=33,header=T,
	stringsAsFactors=F,comment.char='\\')
basevox<-subset(basevox,basevox$Time==0)
mask<-antsImageRead('/home/jeff/Longitudinal/oasis_mask_cx_2mm_crop.nii.gz',3)
basemat<-imagesToMatrix(basevox$InputFile,mask)
m1<-lm(basemat[,1]~basevox$Group+basevox$base_MMSETotal+basevox$base_MMSETimeDiff+
	basevox$base_ageatMRI)
m<-lm(basemat~basevox$Group+basevox$base_MMSETotal+basevox$base_MMSETimeDiff+
	basevox$base_ageatMRI)
mstat<-bigLMStats(m)

# Save out model residuals.
rlist<-matrixToImages(m$residuals,mask)
for (i in 1:length(rlist)) {
	fname<-paste('resid',sprintf('%03d',i),'.nii.gz',sep='')
	antsImageWrite(rlist[[i]],fname)
}
system('fslmerge -t base_resid.nii.gz resid???.nii.gz')
system('rm resid???.nii.gz')

# Estimate spatial auto-correlation in data from residuals using 3dFWHMx.
# system('3dFWHMx -mask oasis_mask_cx_2mm_crop.nii.gz -input base_resid.nii.gz -ACF -out base_resid')
# ACF model parameters for a*exp(-r*r/(2*b*b))+(1-a)*exp(-r/c) plus effective FWHM
# 0.650884  5.22967  23.5627    14.645

# Determine appropriate cluster threshold for cluster-wise alpha=0.05.
#system('3dClustSim -mask oasis_mask_cx_2mm_crop.nii.gz -acf 0.650884 5.22967 23.5627 -pthr 0.001 0.0001 -athr 0.05 0.01 -prefix base_resid_clustsim_acf')

# 3dClustSim -mask oasis_mask_cx_2mm_crop.nii.gz -acf 0.650884 5.22967 23.5627 -pthr 0.001 0.0001 -athr 0.05 0.01 -prefix base_resid_clustsim_acf
# bi-sided thresholding
# Grid: 70x66x90 2.00x2.00x2.00 mm^3 (80256 voxels in mask)
#
# CLUSTER SIZE THRESHOLD(pthr,alpha) in Voxels
# -NN 3  | alpha = Prob(Cluster >= given size)
#  pthr  | .05000 .01000
# ------ | ------ ------
# 0.001000    70.9  112.3
# 0.000100    22.0   39.1
kthresh<-71

# Save out results at clusterwise p<0.05.
for (i in 1:4) {

	# Threshold at voxelwise p<0.001.
	wzero<-which(mstat$beta.pval[i,]>0.001)
	tthresh<-matrix(mstat$beta.t[i,],nrow=1)
	tthresh[wzero]<-0
	lothresh<-sprintf('%.1f',min(abs(tthresh[abs(tthresh)>0])))
	tthresh<-matrixToImages(tthresh,mask)
	g<-sort(unique(basevox$Group))[i+1]
	g<-unlist(strsplit(g,'_'))[2]
	outfile<-paste('base_',g,'_Control_tstat_p001.nii.gz',sep='')
	antsImageWrite(image=tthresh[[1]],file=outfile)
	cmd<-paste('3dmerge -overwrite -dxyz=1 -1clust 1.75 ',kthresh,' -1dindex 0 -1tindex 0 -2thresh -',
		lothresh,' ',lothresh,' -prefix ',gsub('tstat_p001.nii','clust.nii',outfile),' ',
		outfile,sep='')
	system(cmd)
}

baseflist<-c('base_aAD_Control_clust.nii.gz','base_fvAD_Control_clust.nii.gz',
	'base_lvPPA_Control_clust.nii.gz','base_PCA_Control_clust.nii.gz')

for (i in 1:length(baseflist)) {
	f<-baseflist[i]
	cmd<-paste('3dresample -master T_template0_BrainCerebellum_crop.nii.gz -prefix ',
		f,' -inset ',f,' -overwrite',sep='')
	system(cmd)
	cmd<-paste('fslswapdim ',f,' x z -y ',gsub('.nii','_RPI.nii',f))
	system(cmd)
}

nvox<-ncol(mstat$beta.t)

doThis<-FALSE; if (doThis) {
	# Calculate and save out posthoc contrast files.
	vox_base_z<-matrix(0,nrow=nrow(kbase),ncol=nvox)
	vox_base_p<-matrix(Inf,nrow=nrow(kbase),ncol=nvox)
	cat('Starting voxelwise posthoc tests:',base::date(),'\n')
	for (i in 1:nvox) {
	#	cat(i,'/',nvox,'\n')
		m<-lm(basemat[,i]~basevox$Group+basevox$base_MMSETotal+
			basevox$base_MMSETimeDiff+basevox$base_ageatMRI)
		comp <- glht(m, linfct=kbase)
		scomp<-summary(comp,test=adjusted('none'))
		vox_base_z[,i]<-scomp$test$tstat
		vox_base_p[,i]<-scomp$test$pvalues
	}
	connames<-names(scomp$test$tstat)
	connames<-gsub('mean','base',connames)
	connames<-gsub('g0_','',gsub('g1_','',gsub('g2_','',gsub('g3_','',gsub('g4_',
		'',connames)))))

	for (i in 1:length(connames)) {
		cn<-connames[i]
		wzero<-which(vox_base_p[i,]>0.001)
		tthresh<-matrix(vox_base_z[i,],nrow=1)
		tthresh[wzero]<-0
		lothresh<-sprintf('%.1f',min(abs(tthresh[abs(tthresh)>0])))
		tthresh<-matrixToImages(tthresh,mask)
		outfile<-paste(cn,'.nii.gz',sep='')
		f<-gsub('.nii','_clust.nii',outfile)
		antsImageWrite(image=tthresh[[1]],file=outfile)
		cmd<-paste('3dmerge -overwrite -dxyz=1 -1clust 1.75 ',kthresh,' -1dindex 0 -1tindex 0 -2thresh -',
			lothresh,' ',lothresh,' -prefix ',f,' ',
			outfile,sep='')
		system(cmd)
		cmd<-paste('3dresample -master T_template0_BrainCerebellum_crop.nii.gz -prefix ',
			f,' -inset ',f,' -overwrite',sep='')
		system(cmd)
		cmd<-paste('fslswapdim ',f,' x z -y ',gsub('.nii','_RPI.nii',f))
		system(cmd)
	}
	cat('Finished voxelwise posthoc tests:',base::date(),'\n')
}

```

```{r run_vox_lme, eval=TRUE, echo=FALSE, include=TRUE, results='asis',autodep=TRUE}

# Run voxelwise LME on cortical thickness maps.
#system('~/bin/long_lme_ctx_20180525.sh')

# Estimate spatial auto-correlation in data from LME residuals using 3dFWHMx.
# system('3dFWHMx -mask oasis_mask_cx_2mm_crop.nii.gz -input long_lme_ctx_resid_20180530.nii.gz -ACF -out long_lme_ctx_resid_20180530')

# ACF model parameters for a*exp(-r*r/(2*b*b))+(1-a)*exp(-r/c) plus effective FWHM
# 0.602317  5.62269  15.4545    14.7909

# Determine appropriate cluster threshold for cluster-wise alpha=0.05.
# system('3dClustSim -mask oasis_mask_cx_2mm_crop.nii.gz -acf 0.602317 5.62269 15.4545 -pthr 0.001 0.0001 -athr 0.05 0.01 -prefix resid_clustsim_acf_20180530')
# 3dClustSim -mask oasis_mask_cx_2mm_crop.nii.gz -acf 0.602317 5.62269 15.4545 -pthr 0.001 0.0001 -athr 0.05 0.01 -prefix resid_clustsim_acf_20180530
# bi-sided thresholding
# Grid: 70x66x90 2.00x2.00x2.00 mm^3 (80256 voxels in mask)
#
# CLUSTER SIZE THRESHOLD(pthr,alpha) in Voxels
# -NN 3  | alpha = Prob(Cluster >= given size)
#  pthr  | .05000 .01000
# ------ | ------ ------
# 0.001000    69.9  106.6
# 0.000100    21.7   38.0

th<-3.277
k<-70

subbriks<-data.frame(SB=seq(8,52,by=2),Label=c("mean_Control_lvPPA","mean_Control_PCA",
	"mean_Control_aAD","mean_Control_fvAD","mean_PCA_lvPPA","mean_aAD_lvPPA",
	"mean_aAD_PCA","mean_aAD_fvAD","mean_PCA_fvAD","mean_lvPPA_fvAD",
	"time_Control_lvPPA","time_Control_PCA","time_Control_aAD","time_Control_fvAD",
	"time_PCA_lvPPA","time_aAD_lvPPA","time_aAD_PCA","time_aAD_fvAD","time_PCA_fvAD",
	"time_lvPPA_fvAD","base_MMSE","base_age","time_Control"),stringsAsFactors=F)

for (i in 1:nrow(subbriks)) {
	ind<-subbriks$SB[i]
	outf<-paste(subbriks$Label[i],'_clust.nii.gz',sep='')
	cmd<-paste('3dmerge -dxyz=1 -1clust 1.75 ',k,' -1dindex ',ind,' -1tindex ',ind,
		' -2thresh -',th,' ',th,' -prefix ',outf,' long_lme_ctx_20180530+orig -overwrite',sep='')
	system(cmd)
	cmd<-paste('3dresample -master T_template0_BrainCerebellum_crop.nii.gz -prefix ',
		outf,' -inset ',outf,' -overwrite',sep='')
	system(cmd)
	cmd<-paste('fslswapdim ',outf,' x z -y ',gsub('.nii','_RPI.nii',outf))
	system(cmd)
}

th<-2.577
k<-70

for (i in 1:nrow(subbriks)) {
	ind<-subbriks$SB[i]
	outf<-paste(subbriks$Label[i],'_p01_clust.nii.gz',sep='')
	cmd<-paste('3dmerge -dxyz=1 -1clust 1.75 ',k,' -1dindex ',ind,' -1tindex ',ind,
		' -2thresh -',th,' ',th,' -prefix ',outf,' long_lme_ctx_20180530+orig -overwrite',sep='')
	system(cmd)
	cmd<-paste('3dresample -master T_template0_BrainCerebellum_crop.nii.gz -prefix ',
		outf,' -inset ',outf,' -overwrite',sep='')
	system(cmd)
	cmd<-paste('fslswapdim ',outf,' x z -y ',gsub('.nii','_RPI.nii',outf))
	system(cmd)
}

doThis<-FALSE; if (doThis) {
	b<-peaktab[!peaktab$lvPPA.base=='---',c(1:2,7:8)]
	d<-peaktab[!peaktab$lvPPA.time=='---',c(1:2,9:10)]
	o<-peaktab[peaktab$lvPPA.olap %in% 1,c(1:2,11:12)]
	wrs<-d[d$LabelID %in% setdiff(intersect(b$LabelID,d$LabelID),o$LabelID),]
	n<-d[d$LabelID %in% setdiff(d$LabelID,b$LabelID),]

	b<-peaktab[!peaktab$PCA.base=='---',c(1:2,13:14)]
	d<-peaktab[!peaktab$PCA.time=='---',c(1:2,15:16)]
	o<-peaktab[peaktab$PCA.olap %in% 1,c(1:2,17:18)]
	wrs<-d[d$LabelID %in% setdiff(intersect(b$LabelID,d$LabelID),o$LabelID),]
	n<-d[d$LabelID %in% setdiff(d$LabelID,b$LabelID),]

	b<-peaktab[!peaktab$fvAD.base=='---',c(1:2,19:20)]
	d<-peaktab[!peaktab$fvAD.time=='---',c(1:2,21:22)]
	o<-peaktab[peaktab$fvAD.olap %in% 1,c(1:2,23:24)]
	wrs<-d[d$LabelID %in% setdiff(intersect(b$LabelID,d$LabelID),o$LabelID),]
	n<-d[d$LabelID %in% setdiff(d$LabelID,b$LabelID),]

	b<-peaktab[!peaktab$aAD.base=='---',c(1:4)]
	d<-peaktab[!peaktab$aAD.time=='---',c(1:2,5:6)]
	wrs<-d[d$LabelID %in% intersect(b$LabelID,d$LabelID),]
	n<-d[d$LabelID %in% setdiff(d$LabelID,b$LabelID),]
}
```

Exploratory analysis highlighted regions where each phenotype had lower cortical thickness than controls at the time of their initial MRI scan; regions that displayed both initial atrophy as well as atrophy progression over time; and regions without initial atrophy where patients had a higher longitudinal rate of atrophy than controls. Supplementary Table 6 reports the peak t-statistic (local minimum) and volume occupied in each of the Mindboggle cortical regions of interest for contrasts of initial cortical thickness vs. controls, and Supplementary Table 7 similarly reports regional peaks for contrasts of longitudinal rate of change vs. controls. 

At the time of initial MRI, the lvPPA group exhibited lower cortical thickness vs. controls in left middle and superior temporal gyri, our hypothesized disease focus for lvPPA, corroborating ROI volume analysis. Additionally, we observed additional cortical thinning in the left central and parietal opercula; planum temporale; planum polare; and left inferior temporal, fusiform, supramarginal, angular, inferior occipital, and middle occipital gyri. In prefrontal cortex, lvPPA patients had cortical thinning in left anterior insula and frontal operculum as well as left anterior orbital, left medial orbital, and bilateral middle and superior frontal gyri. Focal thinning of left parahippocampal gyrus, which was detected in ROI-based analysis, was observed only at an uncorrected threshold of p<0.01 (Supplementary Figure 2) and did not survive multiple comparisons correction.

Several voxel clusters in which lvPPA patients displayed initial thinning failed to exhibit further progression in longitudinal analysis, including portions of left perisylvian cortex, posterior insula, and inferior occipital and occipital fusiform gyri as well as bilateral middle and superior frontal gyri (Figure 3A, blue regions). The absence of longitudinal progression in these areas suggests that neurodegeneration had reached a plateau during the longitudinal course of these observations. In contrast, a number of left temporal and inferior parietal voxel clusters displayed both initial and longitudinal atrophy, including inferior, middle, and superior temporal gyri; perisylvian cortex; and supramarginal, angular, and anterior fusiform gyri (Figure 3A, green regions). A similar pattern of effects was observed throughout left insular and opercular cortex and in bilateral middle and superior frontal gyri. Finally, longitudinal progression in the absence of initial atrophy occurred throughout bilateral prefrontal, medial parietal, and anterior temporal cortex as well as right inferior parietal/posterior temporal cortex (Figure 3A, red regions), suggesting spread of disease to these areas following patients' initial scans.

PCA patients' initial neocortical atrophy corroborated ROI-based results (Table 2) and additionally indicated homologous atrophy in left precuneus, superior parietal lobule and angular gyrus (Figure 3B). Initial cortical thinning was also observed in bilateral inferior occipital, middle occipital, and posterior cingulate gyri; right cuneus as well as precentral, middle frontal, and superior frontal gyri, and inferior occipitotemporal cortex; and left superior occipital and superior temporal gyri. Areas of initial atrophy that continued to progress longitudinally included bilateral posterior cingulate gyri as well as right precentral gyrus, left precuneus, and left supramarginal and superior temporal gyri. Progressive atrophy was observed in several areas that were not atrophied at the time of initial MRI, including bilateral postcentral gyri, transverse temporal gyri, plana temporalia, central opercula, and temporal poles; left frontal pole, middle/superior frontal gyri, inferior frontal gyrus (pars opercularis and pars triangularis), and middle cingulate gyrus; and right parietal operculum and planum polare.

As in ROI-based analysis, fvAD patients exhibited initial atrophy relative to controls in left anterior insula and middle frontal gyrus, right angular gyrus, and bilateral middle temporal gyri. Additionally, the fvAD group had cortical thinning in left supplementary motor area and supramarginal gyrus as well as right anterior insula, middle frontal gyrus, medial prefrontal cortex, posterior cingulate, and posterior insula. Cortical thinning was also observed bilaterally in the inferior and superior frontal gyri; anterior, middle, and posterior cingulate gyri; inferior and superior temporal gyri; frontal opercula; and temporal poles (Figure 3C). The fvAD group also had initial atrophy relative to controls in the anterior and dorsal portion of right entorhinal cortex, a finding that was not captured by ROI-based analysis. Differences in the rate of atrophy between fvAD patients and controls were limited to the junction of right anterior insula and frontal/central operculum, which also exhibited atrophy relative to controls at initial MRI.

At initial MRI, the aAD group exhibited lower cortical thickness than controls in bilateral middle and posterior cingulate gyri; left angular and supramarginal gyri as well as inferior, middle, and superior temporal gyri; and right frontal lobe areas including supplementary motor cortex and anterior orbital, middle frontal, superior frontal, and medial precentral gyri (Figure 3D). Additionally, we observed cortical thinning in the left parahippocampal gyrus and bilateral entorhinal cortex at p<0.01, uncorrected (Supplementary Figure 2); however, these results were not significant at cluster-wise p<0.05. Regions where aAD patients displayed a faster rate of atrophy over time in the absence of initial cortical thinning included the bilateral insulae, angular gyri, precunei, supramarginal gyri, and superior parietal lobules as well as portions of right prefrontal, lateral temporal, inferior occipito-temporal, and occipital cortex. In the aAD group, regions that differed from controls in rates of longitudinal atrophy did not overlap with regions displaying atrophy at first scan.

\begin{figure}[]
% Voxelwise group differences in cortical thickness.

{\centering \includegraphics[width=7in]{./fig/vox_base_diff.pdf} 

}

\caption[GM mean differences]{Voxelwise differences between patient groups in cortical thickness at time of initial MRI scan. Results are thresholded at voxelwise p<0.001 with a minimum cluster volume of \SI{560}{\micro\litre}, corresponding to a corrected cluster-wise threshold of p<0.05. Warm colors indicate thinner cortical grey matter in the second group than the first; cool colors indicate thinner cortical grey matter in the first group than the second. \label{gm_mean_diff}}

\end{figure}

Figure 4 presents contrasts between patient groups of initial cortical thickness. Relative to aAD, lvPPA patients exhibited more severe atrophy than aAD patients in left fusiform and inferior temporal gyri, consistent with the expected left-lateralization of disease in these patients (Figure 4A). Conversely, lvPPA patients had less atrophy in right entorhinal cortex. Lateralization was also evident in PCA patients, who had lower cortical thickness vs. the lvPPA group throughout right but not left occipitotemporal and parietal cortex (Figure 4D), including the superior parietal lobule as well as the posterior cingulate, fusiform, occipital fusiform, and inferior and superior occipital gyri. PCA patients had greater atrophy than aAD patients in bilateral superior parietal lobules and precunei as well as right fusiform and inferior temporal gyri (Figure 4B); conversely, PCA patients had less severe atrophy than aAD patients in ventromedial prefrontal cortex. In contrast to all other groups, fvAD patients exhibited more severe atrophy in frontal and anterior temporal areas. Relative to aAD patients, the fvAD group had more severe atrophy in right medial superior frontal gyrus, supplementary motor area, and medial orbital gyrus (Figure 4C). Relative to lvPPA patients, the fvAD group had more severe atrophy in right entorhinal cortex, posterior insula, and middle temporal and middle cingulate gyri (Figure 4D). Finally, fvAD patients had more severe atrophy than PCA patients in ventromedial prefrontal regions including anterior cingulate and subcallosal cortex as well as the gyri recti; conversely, fvAD patients had less severe atrophy than PCA patients in left middle and superior occipital gyri. Collectively, these results replicate initial volume differences from ROI-based analysis and highlight additional phenotype-specific areas of atrophy reported in prior studies of lvPPA [@rogalski_aphasic_2016], PCA [@lehmann_global_2012], and fvAD [@whitwell_temporoparietal_2011].

\begin{figure}[]
% Voxelwise group differences in rates of longitudinal GM change.

{\centering \includegraphics[width=7in]{./fig/vox_group_time.pdf} 

}

\caption[GM group x time differences]{Voxelwise differences between patient groups in rates of cortical thinning over time. Image overlays are t-statistic maps for the interaction of each group with time, calculated from linear mixed effects (LME) models and thresholded at voxelwise p<0.001 with a minimum cluster volume of \SI{560}{\micro\litre}, corresponding to a corrected cluster-wise threshold of p<0.05. Warm colors indicate that cortical thinning over time is more rapid in the second group than the first; cool colors indicate that cortical thinning is more rapid in the first group than the second. No differences in rates of cortical thinning were observed between aAD and PCA patients or between lvPPA and PCA patients. \label{gm_group_time}}

\end{figure}

Finally, we assessed group differences in the rate of cortical thinning over time. In general, we found more rapid rates of progressive atrophy in core neocortical regions and also at the margins of core regions associated with each naAD subgroup. Left posterior superior temporal gyrus, which exhibited faster atrophy for lvPPA than aAD patients in ROI-based analysis, demonstrated the same pattern in voxelwise analysis (Figure 5A). FvAD patients exhibited focal differences relative to the other patient groups in prefrontal, parietal, and temporal cortex. Atrophy was faster in fvAD than aAD in right anterior cingulate cortex but more gradual than the aAD group in right supramarginal and middle occipital gyri (Figure 5B). Relative to fvAD patients, lvPPA patients had more rapid atrophy in bilateral inferior and middle occipital gyri as well as left precuneus and right supramarginal gyri (Figure 5C). PCA patients also exhibited more rapid atrophy than fvAD patients posteriorly in right middle occipital gyrus (Figure 5D). In entorhinal cortex and parahippocampal gyri, rates of cortical thinning did not differ in aAD compared to naAD.

\begin{figure}[]
% Voxelwise associations with age and MMSE.

{\centering \includegraphics[width=7in]{./fig/age_mmse.pdf} 

}

\caption[Age and global cognition associations]{Voxelwise associations of cortical thickness with age and MMSE score at initial MRI. Image overlays are t-statistic maps from linear mixed effects (LME) models, thresholded at voxelwise p<0.001 with a minimum cluster volume of \SI{560}{\micro\litre}, corresponding to a corrected cluster-wise threshold of p<0.05. Warm colors indicate that cortical thickness over time is positively associated with each variable; cool colors indicate inverse associations. \label{age_mmse}}

\end{figure}

Independent of group differences in longitudinal atrophy, age at initial MRI was inversely associated with cortical thickness, including large portions of the bilateral medial and anterior temporal lobes, including bilateral temporal poles, entorhinal cortex, parahippocampal gyri, and anterior fusiform gyri; bilateral precentral, postcentral, and medial orbital gyri; and right lingual gyrus, calcarine cortex, cuneus, and middle temporal gyrus (Figure 6, top). Additionally, age was positively associated with cortical thickness in the precuneus, which exhibits greater atrophy in earlier-onset than later-onset Alzheimer's disease [@moller_different_2013]. Associations with baseline MMSE score were distributed throughout all lobes of the brain (Figure 6, bottom).

# Discussion {-}

Few studies to date have addressed the longitudinal spread of disease in naAD. Previous longitudinal imaging studies of non-amnestic Alzheimer's disease have focused on single variants such as lvPPA [@rogalski_progression_2011;@rohrer_patterns_2013] and PCA [@lehmann_global_2012]. The current study compares longitudinal disease progression in multiple clinically-defined naAD phenotypes with autopsy or CSF evidence of Alzheimer's disease pathology. The longitudinal design allowed us to investigate not only areas of initial atrophy in each phenotype, but how atrophy progressed as a result of spreading pathology. Furthermore, we asked whether longitudinal rates of neurodegeneration differed across patient groups in phenotype-specific disease foci, a finding which could at least partially account for each phenotype's characteristic disease distribution. Below, we review areas of initial and progressive atrophy in each phenotype as well as differences between groups in the longitudinal progression of atrophy in the neocortex and MTL. Collectively, the current results provide valuable support for the hypothesis that pathology originates in and emanates from phenotype-specific neocortical areas rather than the MTL in naAD and corroborate previous autopsy studies of hippocampal-sparing Alzheimer's disease that have been limited by sparse clinical characterization of patients and the cross-sectional nature of postmortem studies  [@ferreira_distinct_2017;@murray_neuropathologically_2011;@giannakopoulos_alzheimers_1994].

Contrasts relative to controls illustrate the anatomical progression of disease in each patient group. In all phenotypes, some brain areas exhibited significant atrophy at the initial scan without a temporal progression. Following the logic of pathology studies, we reasoned that such regions are likely involved early in the disease process; because they exhibited minimal progression over the study period, we infer that these regions have already undergone advanced neurodegeneration. We also observed other areas that exhibited both atrophy at initial MRI and significant change over time. These areas represent an important window on progressive neurodegeneration, where atrophy is ongoing and has not yet reached a plateau. Finally, areas that demonstrated change over time but whose initial values did not differ from controls may have begun to degenerate only later in the disease process.

ROI-based and voxelwise analyses provided converging and complementary evidence regarding atrophy patterns at initial MRI. Overall, lvPPA patients exhibited strong left lateralization of atrophy, consistent with prior studies [@phillips_neocortical_2018;@rogalski_aphasic_2016]. In ROI-based analysis, lvPPA patients had neocortical volume loss relative to controls in left middle and superior temporal gyri, anterior insula, and middle frontal gyrus. Voxelwise analysis corroborated these findings and indicated additional atrophy in left occipitotemporal and posterior temporal/inferior parietal cortex as well as left orbitofrontal cortex and bilateral superior frontal gyri. Additionally, ROI-based analysis indicated that lvPPA patients had volume loss in left hippocampus and parahippocampal gyrus. In the PCA group, ROI-based analysis indicated initial atrophy in right hippocampus, precuneus, superior parietal lobule, and angular gyrus as well as in bilateral middle temporal gyri. Voxelwise analysis indicated that this atrophy extended to homologous left-hemisphere areas as well as bilateral occipital cortex; right cuneus, prefrontal cortex, and inferior occipitotemporal cortex; and left superior temporal gyrus. In the fvAD group, ROI-based analysis indicated initial volume loss in left anterior insula and middle frontal gyrus, right angular gyrus, and bilateral middle temporal gyri. Voxelwise analysis corroborated these findings and indicated additional atrophy throughout bilateral medial and lateral prefrontal cortex. Additionally, fvAD patients exhibited initial cortical thinning in right posterior cingulate gyrus, entorhinal cortex, and anterior and posterior insula; left supramarginal gyrus; and bilateral temporal lobes, including the temporal poles, perisylvian cortex, and inferior and superior temporal gyri. Finally, ROI-based analysis indicated that the aAD group had initial volume loss relative to controls in all six MTL regions but in none of the other regions tested. Voxelwise cortical thickness was lower in the aAD group than in controls in bilateral middle and posterior cingulate gyri; left inferior parietal and lateral temporal cortex; and right medial and lateral prefrontal areas. Cortical thinning in the left parahippocampal gyrus and bilateral entorhinal cortex was observed at a lenient statistical threshold but did not survive multiple comparisons correction. As discussed previously, this discrepancy between ROI-based and voxelwise results may indicate that the precise focus of atrophy varied across patients in these structures, or it may result from the difficulty of segmenting MTL structures. The presence of some MTL atrophy at baseline imaging in lvPPA and PCA differs from our previous cross-sectional analysis of atrophy in naAD [@phillips_neocortical_2018], which concluded that MTL areas were not significantly atrophied relative to controls in early disease. This discrepancy may be due to differences in analysis approaches: while our previous study used a non-parametric, frequency-based algorithm inspired by pathological staging studies, the current study uses a more traditional parametric statistical approach. Additional work is needed to resolve these differences. Nevertheless, both ROI-based and voxelwise analyses showed that lvPPA patients had significantly less atrophy than aAD patients in MTL regions. Overall, cross-sectional comparisons of atrophy at initial MRI largely corroborated prior conclusions regarding the anatomical origin of disease in each phenotype [@phillips_neocortical_2018;@ossenkoppele_atrophy_2015;@xia_association_2017]. 

A subset of regions that exhibited initial atrophy failed to exhibit further longitudinal change during the study period. These regions may represent areas where atrophy has slowed or ceased as the volume of remaining tissue decreases [@sabuncu_dynamics_2011;@schuff_nonlinear_2012], even as continued progression is observed in other regions. This pattern of effects is likely to indicate foci of earliest disease in each phenotype. In ROI-based analysis of the PCA group, all regions that exhibited initial atrophy also displayed significant longitudinal change; however, voxelwise analysis revealed clusters of early, non-progressing atrophy in bilateral calcarine cortex, cunei, occipital poles, and inferior, middle, and superior occipital gyri; right superior parietal lobule, supplementary motor cortex, and right fusiform, lingual, and occipital fusiform gyri; and left inferior temporal gyrus. These areas are largely consistent with the suspected anatomical origin of disease in PCA based on prior studies [@lehmann_global_2012;@tang-wai_clinical_2004]. In fvAD, nearly all of the areas that exibited initial atrophy failed to display longitudinal progression, including large portions of bilateral ventral and dorsal medial and lateral prefrontal cortex as well as more anterior portions of the temporal lobes, as described above; these findings support the hypothesis that patients with behavioral and executive function deficits due to underlying Alzheimer's disease have disease originating in the frontal lobe. In aAD, initial atrophy without subsequent progression was observed in left entorhinal cortex (Table 2), corresponding to early Braak stages [@braak_neuropathological_1991]. Initial, non-progressing atrophy was also observed in bilateral middle and posterior cingulate gyri; left inferior, middle, and superior temporal gyri; right anterior orbital and medial precentral gyri; and right supplementary motor cortex (Figure 3, Supplementary Table 6). In the context of aAD, many of these neocortical areas are unlikely to reflect early involvement in the disease process, since they correspond to Braak stages of V--VI in aAD; we may have failed to detect longitudinal change in these areas due to statistical power issues or because the areas in question are on the lower end of normal variation in cortical thickness [@fischl_measuring_2000;@sowell_longitudinal_2004;@von_economo_cytoarchitectonics_1929] and exhibit smaller changes over time. In contrast to other groups, lvPPA patients exhibited longitudinal progression in some form in all areas that displayed initial atrophy.

Many of the areas that exhibited initial atrophy relative to controls also exhibited longitudinal change, suggesting that neurodegeneration had begun prior to initial MRI and did not reach a plateau during the study period. In lvPPA, these areas included frontal, central, and parietal opercula; inferior, middle, and superior temporal gyri; angular, supramarginal, fusiform, and middle occipital gyri; anterior insula; planum temporale (Figure 3, Supplementary Tables 6--7); and left hippocampus and parahippocampal gyrus (Table 2). In several larger ROIs, voxel clusters that exhibited cortical thinning at initial MRI did not exhibit subsequent progression, but adjacent clusters within the same ROI did. Thus, the lvPPA group showed early atrophy within the left middle frontal gyrus that remained stable over the follow-up period (Figure 3, top row, blue), but atrophy appeared to spread to neighboring voxels within the middle frontal gyrus (Figure 3, red); ROI analysis corroborated longitudinal change in left middle frontal gyrus among lvPPA patients. Similar within-region spread of atrophy was observed in bilateral prefrontal cortex as well as left posterior insula, perisylvian cortex, occipital fusiform gyrus, and orbitofrontal cortex. In the PCA group, ROI-based analysis indicated overlapping effects of initial and progressive atrophy in bilateral middle temporal gyri as well as right hippocampus, precuneus, superior parietal lobule, and angular gyrus. Voxelwise analysis corroborated each of these findings and additionally indicated overlapping initial and longitudinal atrophy in bilateral supramarginal and posterior cingulate gyri; left parietal operculum as well as angular and superior temporal gyri; and right middle cingulate, middle frontal, and precentral gyri. Within-region spread of atrophy was additionally observed in voxelwise analysis throughout the right temporal lobe and left superior parietal cortex. In the fvAD group, ROI-based analysis indicated both initial and longitudinal atrophy in bilateral middle temporal gyri and left anterior insula. In voxelwise analysis, the overlap between initial and longitudinal atrophy was restricted to right anterior insula, frontal operculum, and central operculum; the left anterior insula was also detected at a more lenient statistical threshold (Supplementary Figure 2). ROI-based analysis in the aAD group indicated that five of six MTL regions (excluding left entorhinal cortex) exhibited both initial and longitudinal atrophy relative to controls. Additionally, voxelwise analysis suggested within-region spread of atrophy in left angular and supramarginal gyri as well as right middle and superior frontal gyri, as evidenced clusters of initial atrophy (but no longitudinal change) located proximally to clusters that exhibited longitudinal change (but no initial atrophy).

Finally, both ROI-based and voxelwise analysis detected several areas in each phenotype that displayed longitudinal progression relative to controls in the absence of initial atrophy. This profile of effects reflects brain areas that developed significant atrophy after the start of the study period and are likely to represent areas of later involvement in each phenotype. In lvPPA, these areas of latest involvement were widespread and included bilateral entorhinal cortex as well as right hippocampus and parahippocampal gyrus (Table 2); and bilateral atrophy in the temporal poles, inferior frontal gyri, occipitotemporal cortex, precunei, posterior cingulate gyri, superior parietal lobules, precentral gyri, and medial aspects of the superior frontal gyri (Figure 3, Supplementary Table 7). Additional areas of later atrophy included the left frontal pole and postcentral gyrus. Areas of later right hemisphere atrophy in lvPPA included perisylvian and lateral temporal cortex, middle and angular and supramarginal gyri, ventral occipitotemporal areas, and inferior frontal gyrus; these findings appear to represent spread of disease from homologous left hemisphere areas affected early in the disease process, as reported in previous longitudinal imaging of PPA [@rogalski_progression_2011;@rohrer_patterns_2013]. In the PCA group, ROI-based analysis indicated later longitudinal atrophy in right entorhinal cortex and parahipppocampal gyrus as well as left hippocampus; left superior temporal and middle frontal gyri; and right supramarginal gyrus (Table 2). Voxelwise analysis additionally indicated later atrophy in bilateral postcentral gyri, perisylvian cortex, central opercula, and temporal poles; left middle cingulate, inferior frontal (pars opercularis), precentral, and superior frontal gyri; and right parietal operculum (Figure 3, Supplementary Table 7). These results are consistent with the spread of PCA into more anterior portions of the temporal lobes and into prefrontal cortex [@lehmann_global_2012]. In the fvAD group, ROI-based analysis indicated later atrophy in bilateral entorhinal cortex as well as left superior temporal gyrus, while voxelwise analysis indicated later cortical thinning in the right anterior insula and frontal operculum; discrepancies between ROI-based and voxelwise results in this case may result from differences in the sensitivity of the two methods for atrophy detection or to the difficulty of segmenting MTL regions, as previously discussed. In the aAD group, ROI-based analysis indicated later atrophy in bilateral middle temporal gyri, left middle frontal and superior temporal gyri, and right angular gyrus, supramarginal gyrus, precuneus, and superior parietal lobule (Table 2). Voxelwise results similarly indicated clusters of longitudinal cortical thinning in each of these areas (Figure 3, Supplementary Figure 2) and also indicated clusters of later atrophy in bilateral anterior and posterior insulae, and posterior orbital gyri; left frontal operculum, inferior frontal gyrus (pars opercularis), precuneus, and superior parietal lobule; and right central operculum, parietal operculum, cuneus, temporal pole, lateral temporal cortex (inferior and superior temporal gyri), and occipitotemporal areas (fusiform, middle occipital, and superior occipital gyri).  These findings in aAD are consistent with inferences about disease staging in pathology studies of aAD [@braak_neuropathological_1991], and autopsy studies of regional pathologic burden in naAD [@josephs_progressive_2008] would help verify our findings in these less common variants of Alzheimer's disease. Collectively, areas exhibiting longitudinal atrophy relative to controls in the absence of initial cross-sectional differences provide a window onto how disease spreads within each phenotype in more advanced disease.

Direct comparisons between patient groups further corroborated established phenotype-specific patterns of atrophy [@ossenkoppele_atrophy_2015;@phillips_neocortical_2018], including sparing of the MTL relative to aAD [@murray_neuropathologically_2011]. At initial MRI, ROI analysis showed that lvPPA patients exhibited significantly greater GM volume than aAD patients in bilateral entorhinal cortex, right hippocampus, and right parahippocampal gyrus (Figure 1). FvAD patients similarly exhibited greater GM volume than the aAD group in left entorhinal cortex, and PCA patients exhibited marginally greater volumes than aAD patients in left hippocampus and entorhinal cortex (both p<0.1, corrected). These results indicate relative sparing of MTL structures in naAD and demonstrate that aAD patients had undergone more extensive atrophy in MTL areas than naAD patients prior to their initial MRI scans. In the neocortex, ROI analysis showed that lvPPA patients exhibited reduced GM volumes relative to other patient groups in left lateral temporal cortex and relative to PCA patients in left anterior insula. Voxelwise results additionally indicated that lvPPA patients had lower initial cortical thickness than aAD patients in left inferior temporal and fusiform gyri. ROI analysis in PCA patients, in turn, indicated lower GM volumes than lvPPA and aAD patients in the right precuneus and superior parietal lobule and lower volumes than the lvPPA group in right angular and middle temporal gyri. Voxelwise analysis provided converging evidence for a right parietal disease focus in the current PCA sample: initial cortical thickness was lower in PCA relative to both aAD and lvPPA patients in both right occipitotemporal (fusiform, occipital fusiform, and middle and superior occipital gyri) and parietal (precuneus and superior parietal lobule) areas. ROI analysis in fvAD patients indicated more severe initial atrophy than PCA patients in left middle frontal gyrus; greater atrophy relative to lvPPA patients in right angular and middle temporal gyri; and atrophy relative to aAD patients in the right superior parietal lobule. Voxelwise analysis provided converging evidence that fvAD patients had more severe atrophy in medial prefrontal and right anterior temporal areas than other patient groups; the finding of right temporal degeneration, in particular, is consistent with patients' behavioral symptoms [@seeley_natural_2005]. 

The current study also reports novel comparisons of longitudinal atrophy rates in naAD and aAD. LvPPA patients had more rapid atrophy than aAD patients in left superior temporal (Figure 2) and supramarginal gyri (Figure 5); these differential rates of change are independent of baseline atrophy. Rates of atrophy also differed between fvAD and other phenotypes along an anterior-posterior axis. ROI-based analysis indicated more rapid atrophy for fvAD than lvPPA and PCA patients in left anterior insula; and in voxelwise analysis, fvAD patients had more rapid atrophy than aAD patients in right anterior cingulate cortex. Conversely, fvAD patients exhibited more gradual volume loss than PCA and aAD patients in right precuneus and relative to the aAD group in right supramarginal gyrus (Figure 2); additionally, fvAD patients had more gradual cortical thinning than all other groups in right angular and middle occipital gyri and relative to aAD and lvPPA in right supramarginal gyrus. These neocortical differences may contribute to the rapid, domain-specific clinical progression observed in naAD [@lam_clinical_2013]. Contrary to hypotheses, we observed more rapid neocortical atrophy for aAD than naAD patients in multiple areas, including right supramarginal gyrus (relative to lvPPA and fvAD patients) and right middle temporal gyrus (relative to lvPPA and PCA patients). While we had not hypothesized that these two regions would have specific involvement in aAD, functional connectivity between the hippocampus and each is disrupted in aAD [@park_functional_2017]. Collectively, longitudinal analysis demonstrates that rates of neurodegeneration vary between clinical presentations of Alzheimer's disease as well as between brain areas. These results underscore the importance of examining anatomical disease markers that are specifically associated with patients' primary cognitive deficits rather than simplified measures such as mean cortical thickness or volume.

The importance of phenotype-specific ROIs in our hypothesis-driven analysis is validated by associations between GM volume change in these regions and longitudinal neuropsychological performance. In the memory domain, bilateral hippocampi were the most sensitive anatomical markers of memory change; parahippocampal and entorhinal areas showed lesser but still significant associations. In the language domain, patients' declines in letter fluency, a measure that requires guided retrieval of lexical representations, were associated with reductions in left middle temporal gyrus volume. Moreover, forward digit span had robust associations with left temporal volume change; we have previously reported this measure as a sensitive behavioral marker of lvPPA [@giannini_clinical_2017]. The 4-point speech scale administered as part of the PBAC exhibited no associations with longitudinal GM atrophy, likely due to the limited range of this scale; furthermore, this speech scale was developed to be sensitive to the entire range of deficits found in PPA and is not specifically tuned to deficits found in lvPPA [@libon_philadelphia_2011]. Decline in visuospatial function was associated with volume loss in several right-hemisphere parietal and temporal areas. Finally, decline in reverse digit span, a measure of working memory and executive function, was associated with ROIs specific to the fvAD group (left middle frontal and right middle temporal gyri).

In the MTL, we had expected naAD patients to exhibit slower rates of atrophy than aAD patients, providing a potential explanation for their relative hippocampal sparing and preserved memory. However, although naAD patients---particularly lvPPA cases---had less MTL atrophy at initial MRI than aAD patients, rates of change largely did not differ by group. In fact, in the left entorhinal cortex, fvAD patients displayed more rapid atrophy than aAD patients, a finding which we attribute to slowing of left entorhinal atrophy in aAD due to its already-severe atrophy at initial MRI. The absence of hypothesized differences in rates of MTL atrophy suggests that MTL sparing in naAD takes the form of a delayed onset of MTL atrophy rather than a similar onset with a different rate of change. Thus, MTL sparing in autopsy studies of Alzheimer's disease patients with non-amnestic presentations [@murray_neuropathologically_2011] could be due to later involvement in naAD than in aAD rather than to a slower rate of atrophy. Moreover, secondary analysis found that age effects did not differ between aAD and naAD, suggesting that naAD patients are not protected from normal age-associated degeneration of the MTL [@apostolova_hippocampal_2012]. In all six MTL ROIs tested---but no other ROIs---higher age at the initial scan was associated with more severe atrophy. Voxelwise analysis provided converging evidence for robust age effects throughout the medial and anterior temporal lobes. These findings are consistent with previous studies showing that hippocampal-sparing Alzheimer's disease cases have a younger age at onset [@murray_neuropathologically_2011]. These results are also in line with a recent study of Eckerstr&ouml;m and colleagues [@eckerstrom_similar_2018], where early- (age<65) and late-onset Alzheimer's disease (age>65) showed no differences in hippocampal and cortical atrophy after correcting for age. Finally, such age effects may help explain the occurrence of cases with atypical clinical phenotypes but typical distributions of Alzheimer's disease neuropathology at autopsy [@boon_neuroinflammation_2018;@murray_neuropathologically_2011]. Collectively, these findings suggest that the focal MTL atrophy in aAD may be at least partially due to age effects, independently of clinical phenotype [@van_de_pol_hippocampal_2006}]. More longitudinal studies comparing aAD and naAD are necessary to address this theory, which raises interesting questions about the association between aging and regional susceptibility to disease. Although poor tolerance of MRI makes it difficult to image patients with advanced disease, such late-stage imaging would be useful in determining whether patients with an initial non-amnestic syndrome eventually exhibit spread of atrophy to MTL.

Strengths of the current study include a novel comparison of longitudinal anatomical changes in multiple clinically-defined naAD phenotypes using both a priori ROI-based and whole-brain voxelwise analyses. We sought to ensure the comparability of the heterogeneous patient groups included here by controlling for demographic and clinical characteristics both during sample selection and in statistical analysis. However, one major limitation was the inability to evaluate non-linear atrophy progression in Alzheimer's disease: prior evidence suggests that an initial acceleration due to spreading cumulative damage is followed by a deceleration due to the reduction of intact tissue [@sabuncu_dynamics_2011;@schuff_nonlinear_2012]. Such non-linearities complicate study design and interpretation in ways that may not be fully addressed by equating patient groups for chronological age and estimated disease duration: for example, in the current study, it is possible that areas of early atrophy in each phenotype (i.e., those exhibiting atrophy at initial MRI) have entered the deceleration phase, while for other phenotypes the same regions may have been imaged during the acceleration phase. Investigating longitudinal change in earlier-stage patients may allow us to observe a more complete trajectory of neurodegeneration, and including a minimum of 3--4 imaging timepoints may allow us to discriminate between linear, quadratic, and sigmoid models of neurodegeneration. Additionally, the current study was limited in its ability to investigate associations with the APOE genotype or other genetic risk modifiers for Alzheimer's disease. We found that APOE $\epsilon4$ allele counts added little predictive power to our imaging models after accounting for group effects; however, continued study of the APOE genotype and other genetic risk modifiers in naAD remains an important research aim. Finally, future studies should include patients with corticobasal syndrome due to underlying Alzheimer's disease pathology; insufficient longitudinal data prevented us from including this uncommon naAD phenotype in the current study.

Understanding the neuropathological and clinical heterogeneity of Alzheimer's disease is crucial to understanding the mechanisms of its progression. The current results indicate that the onset and rate of neocortical atrophy varies by region and phenotype in naAD, reflecting longitudinal clinical progression. Moreover, the rate of MTL atrophy in naAD appears to be similar to that found in aAD, although onset of MTL atrophy in naAD is considerably delayed.

```{r author_table, eval=TRUE, results='asis',include=FALSE}
# Author contributions {-}

#\captionsetup{labelformat=empty}
# Add in a table of authors and their roles here.

roles<-data.frame(Name=c('Fulvio Da Re\\*','Jeffrey S. Phillips\\*','David J. Irwin',
	'Corey T. McMillan','Sanjeev N. Vaishnavi','Sharon X. Xie','Edward B. Lee',
	'Leslie M. Shaw','John Q. Trojanowski','David A. Wolk','Murray Grossman'),
	Location=c('University of Milano-Bicocca, Milan',
	rep('University of Pennsylvania, Philadelphia',10)),Role=rep('Author',11),
	Contribution=c('\\*Contributed equally; conceptualized the study; reviewed patient histories; drafted the manuscript for intellectual content; interpreted and analyzed data','\\*Contributed equally; conceptualized the study; analyzed and interpreted the data; drafted the manuscript for intellectual content', 'Diagnosed patients; reviewed patient histories; revised the manuscript for intellectual content','Interpreted data; revised the manuscript for intellectual content','Diagnosed patients; revised the manuscript for intellectual content', 'Advised on statistical methods; interpreted the data; revised the manuscript for intellectual content', 'Performed neuropathology examinations','Performed CSF biomarker quantification','Performed neuropathology examinations; revised the manuscript for intellectual content','Diagnosed patients; interpreted the data; revised the manuscript for intellectual content','Conceptualized the study; interpreted the data; diagnosed patients; reviewed patient histories; revised the manuscript for intellectual content')) 

xroles<-xtable(roles,
	caption='Author roles',
	align=rep('l',ncol(roles)+1),digits=rep(0,ncol(roles)+1))
print(xroles,comment=FALSE,caption.placement='top',floating=T,include.rownames=F,
	sanitize.text.function=identity)

```

# Acknowledegments {-}
The authors would like to thank Dr. Valeria Isella and Dr. Carlo Ferrarese for their valuable feedback on this project.

# Funding {-}
This work was supported by grants from the Alzheimer's Association (AARF-16-443681), National Institutes of Health (AG017586, AG010124, AG043503, and NS088341), BrightFocus Foundation (A2016244S), Dana Foundation, Newhouse Foundation, Wyncote Foundation, Arking Family Foundation, and the Italian Ministry of Education, University, and Research.

# Competing interests {-}
All authors report that they have no competing interests to disclose.

# References {-}

